2023-01-27 16:32:22 UTC | CORE | WARN | (pkg/util/log/log.go:618 in func1) | Agent configuration relax permissions constraint on the secret backend cmd, Group can read and exec
2023-01-27 16:32:22 UTC | CORE | INFO | (pkg/util/log/log.go:590 in func1) | Features detected from environment: kubernetes,docker
2023-01-27 16:32:22 UTC | CORE | INFO | (pkg/runtime/runtime.go:27 in func1) | runtime: final GOMAXPROCS value is: 5
2023-01-27 16:32:22 UTC | CORE | INFO | (cmd/agent/subcommands/run/command.go:246 in startAgent) | Starting Datadog Agent v7.41.1
2023-01-27 16:32:24 UTC | CORE | INFO | (cmd/agent/subcommands/run/command.go:309 in startAgent) | Hostname is: minikube
2023-01-27 16:32:24 UTC | CORE | INFO | (pkg/workloadmeta/store.go:340 in startCandidates) | workloadmeta collector "cloudfoundry-container" could not start. error: component workloadmeta-cloudfoundry-container is disabled: Agent is not running on CloudFoundry
2023-01-27 16:32:24 UTC | CORE | INFO | (pkg/workloadmeta/store.go:340 in startCandidates) | workloadmeta collector "containerd" could not start. error: component workloadmeta-containerd is disabled: Agent is not running on containerd
2023-01-27 16:32:24 UTC | CORE | INFO | (pkg/util/kubernetes/kubelet/kubelet_client.go:285 in checkKubeletConnection) | Successful configuration found for Kubelet, using URL: https://192.168.49.2:10250
2023-01-27 16:32:24 UTC | CORE | INFO | (pkg/workloadmeta/store.go:337 in startCandidates) | workloadmeta collector "kubelet" started successfully
2023-01-27 16:32:24 UTC | CORE | INFO | (pkg/workloadmeta/store.go:340 in startCandidates) | workloadmeta collector "podman" could not start. error: component workloadmeta-podman is disabled: Podman not detected
2023-01-27 16:32:24 UTC | CORE | INFO | (pkg/workloadmeta/store.go:340 in startCandidates) | workloadmeta collector "cloudfoundry-vm" could not start. error: component workloadmeta-cloudfoundry-vm is disabled: Agent is not running on CloudFoundry
2023-01-27 16:32:24 UTC | CORE | INFO | (pkg/workloadmeta/store.go:337 in startCandidates) | workloadmeta collector "docker" started successfully
2023-01-27 16:32:24 UTC | CORE | INFO | (pkg/workloadmeta/store.go:340 in startCandidates) | workloadmeta collector "ecs_fargate" could not start. error: component workloadmeta-ecs_fargate is disabled: Agent is not running on Fargate
2023-01-27 16:32:24 UTC | CORE | INFO | (pkg/api/security/security.go:189 in getClusterAgentAuthToken) | Using configured cluster_agent.auth_token
2023-01-27 16:32:24 UTC | CORE | ERROR | (pkg/workloadmeta/collectors/internal/kubemetadata/kubemetadata.go:73 in Start) | Could not initialise the communication with the cluster agent: temporary failure in clusterAgentClient, will retry later: "https://10.104.229.207:5005/version" is unavailable: Get "https://10.104.229.207:5005/version": dial tcp 10.104.229.207:5005: connect: connection refused
2023-01-27 16:32:24 UTC | CORE | INFO | (pkg/workloadmeta/store.go:154 in Start) | workloadmeta store initialized successfully
2023-01-27 16:32:24 UTC | CORE | INFO | (pkg/collector/python/init.go:322 in resolvePythonExecPath) | Using '/opt/datadog-agent/embedded' as Python home
2023-01-27 16:32:24 UTC | CORE | INFO | (pkg/collector/python/init.go:389 in Initialize) | Initializing rtloader with Python 3 /opt/datadog-agent/embedded
2023-01-27 16:32:24 UTC | CORE | INFO | (pkg/tagger/collectors/workloadmeta_main.go:115 in stream) | workloadmeta tagger collector started
2023-01-27 16:32:24 UTC | CORE | INFO | (pkg/collector/python/datadog_agent.go:127 in LogMessage) | - | (ddyaml.py:143) | monkey patching yaml.load...
2023-01-27 16:32:24 UTC | CORE | INFO | (pkg/collector/python/datadog_agent.go:127 in LogMessage) | - | (ddyaml.py:147) | monkey patching yaml.load_all...
2023-01-27 16:32:24 UTC | CORE | INFO | (pkg/collector/python/datadog_agent.go:127 in LogMessage) | - | (ddyaml.py:151) | monkey patching yaml.dump_all... (affects all yaml dump operations)
2023-01-27 16:32:24 UTC | CORE | INFO | (pkg/collector/collector.go:56 in NewCollector) | Embedding Python 3.8.14 (default, Dec 20 2022, 16:57:03) [GCC 5.4.0 20160609]
2023-01-27 16:32:24 UTC | CORE | INFO | (pkg/autodiscovery/providers/config_reader.go:172 in read) | Searching for configuration files at: /etc/datadog-agent/conf.d
2023-01-27 16:32:24 UTC | CORE | INFO | (pkg/autodiscovery/providers/config_reader.go:248 in collectEntry) | Skipping 'auto_conf.yaml' for integration 'etcd'
2023-01-27 16:32:25 UTC | CORE | INFO | (pkg/autodiscovery/providers/config_reader.go:248 in collectEntry) | Skipping 'auto_conf.yaml' for integration 'kube_controller_manager'
2023-01-27 16:32:25 UTC | CORE | INFO | (pkg/autodiscovery/providers/config_reader.go:248 in collectEntry) | Skipping 'auto_conf.yaml' for integration 'kube_scheduler'
2023-01-27 16:32:25 UTC | CORE | INFO | (pkg/autodiscovery/providers/config_reader.go:248 in collectEntry) | Skipping 'auto_conf.yaml' for integration 'kubernetes_state'
2023-01-27 16:32:25 UTC | CORE | INFO | (pkg/autodiscovery/providers/config_reader.go:172 in read) | Searching for configuration files at: /opt/datadog-agent/bin/agent/dist/conf.d
2023-01-27 16:32:25 UTC | CORE | WARN | (pkg/autodiscovery/providers/config_reader.go:176 in read) | Skipping, open /opt/datadog-agent/bin/agent/dist/conf.d: no such file or directory
2023-01-27 16:32:25 UTC | CORE | INFO | (pkg/autodiscovery/providers/config_reader.go:172 in read) | Searching for configuration files at: 
2023-01-27 16:32:25 UTC | CORE | WARN | (pkg/autodiscovery/providers/config_reader.go:176 in read) | Skipping, open : no such file or directory
2023-01-27 16:32:25 UTC | CORE | INFO | (pkg/config/autodiscovery/autodiscovery.go:95 in DiscoverComponentsFromEnv) | Adding KubeContainer provider from environment
2023-01-27 16:32:25 UTC | CORE | INFO | (pkg/config/autodiscovery/autodiscovery.go:117 in DiscoverComponentsFromEnv) | Adding Kubelet listener from environment
2023-01-27 16:32:25 UTC | CORE | WARN | (pkg/autodiscovery/providers/endpointschecks.go:55 in NewEndpointsChecksConfigProvider) | Cannot get dca client: temporary failure in clusterAgentClient, will retry later: try delay not elapsed yet
2023-01-27 16:32:25 UTC | CORE | INFO | (pkg/autodiscovery/autoconfig.go:310 in initListenerCandidates) | environment listener successfully started
2023-01-27 16:32:25 UTC | CORE | INFO | (pkg/autodiscovery/autoconfig.go:310 in initListenerCandidates) | kubelet listener successfully started
2023-01-27 16:32:25 UTC | CORE | INFO | (pkg/autodiscovery/listeners/workloadmeta.go:134 in Listen) | ad-kubeletlistener initialized successfully
2023-01-27 16:32:25 UTC | CORE | INFO | (pkg/autodiscovery/listeners/environment.go:65 in createServices) | Listener created docker service from environment
2023-01-27 16:32:25 UTC | CORE | INFO | (pkg/autodiscovery/listeners/environment.go:65 in createServices) | Listener created kubelet service from environment
2023-01-27 16:32:25 UTC | CORE | INFO | (pkg/autodiscovery/listeners/environment.go:72 in createServices) | Listener created container service from environment
2023-01-27 16:32:25 UTC | CORE | INFO | (pkg/api/security/security.go:250 in saveAuthToken) | Wrote auth token
2023-01-27 16:32:25 UTC | CORE | INFO | (pkg/api/security/security.go:144 in fetchAuthToken) | Saved a new authentication token to /etc/datadog-agent/auth_token
2023-01-27 16:32:25 UTC | CORE | INFO | (cmd/agent/subcommands/run/command.go:360 in startAgent) | GUI server port -1 specified: not starting the GUI.
2023-01-27 16:32:25 UTC | CORE | INFO | (pkg/forwarder/forwarder.go:233 in NewDefaultForwarder) | Retry queue storage on disk is disabled
2023-01-27 16:32:25 UTC | CORE | INFO | (pkg/aggregator/time_sampler.go:46 in NewTimeSampler) | Creating TimeSampler #0
2023-01-27 16:32:25 UTC | CORE | INFO | (pkg/forwarder/forwarder.go:352 in Start) | Forwarder started, sending to 1 endpoint(s) with 1 worker(s) each: "https://7-41-1-app.agent.datadoghq.com" (1 api key(s))
2023-01-27 16:32:25 UTC | CORE | INFO | (cmd/system-probe/config/config.go:119 in Merge) | no config exists at /etc/datadog-agent/system-probe.yaml, ignoring...
2023-01-27 16:32:25 UTC | CORE | WARN | (pkg/secrets/secrets.go:50 in Init) | Agent configuration relax permissions constraint on the secret backend cmd, Group can read and exec
2023-01-27 16:32:25 UTC | CORE | INFO | (pkg/collector/runner/runner.go:92 in ensureMinWorkers) | Runner 1 added 4 workers (total: 4)
2023-01-27 16:32:25 UTC | CORE | INFO | (pkg/dogstatsd/listeners/uds_common.go:146 in Listen) | dogstatsd-uds: starting to listen on /var/run/datadog/dsd.socket
2023-01-27 16:32:25 UTC | CORE | INFO | (pkg/dogstatsd/listeners/udp.go:95 in Listen) | dogstatsd-udp: starting to listen on [::]:8125
2023-01-27 16:32:25 UTC | CORE | INFO | (pkg/logs/client/http/destination.go:376 in CheckConnectivity) | Checking HTTP connectivity...
2023-01-27 16:32:25 UTC | CORE | INFO | (pkg/logs/client/http/destination.go:382 in CheckConnectivity) | Sending HTTP connectivity request to https://agent-http-intake.logs.datadoghq.com/api/v2/logs...
2023-01-27 16:32:25 UTC | CORE | INFO | (pkg/logs/client/http/destination.go:387 in CheckConnectivity) | HTTP connectivity successful
2023-01-27 16:32:25 UTC | CORE | INFO | (pkg/logs/logs.go:121 in start) | Starting logs-agent...
2023-01-27 16:32:25 UTC | CORE | INFO | (pkg/logs/logs.go:131 in start) | logs-agent started
2023-01-27 16:32:25 UTC | CORE | INFO | (pkg/autodiscovery/config_poller.go:168 in collectOnce) | file provider: collected 52 new configurations, removed 0
2023-01-27 16:32:25 UTC | CORE | INFO | (pkg/logs/internal/launchers/container/launcher.go:92 in run) | Starting Container launcher
2023-01-27 16:32:25 UTC | CORE | INFO | (pkg/collector/scheduler/scheduler.go:92 in Enter) | Scheduling check coredns with an interval of 15s
2023-01-27 16:32:25 UTC | CORE | INFO | (pkg/collector/python/datadog_agent.go:127 in LogMessage) | unknown | (base_check.py:124) | Couldn't connect to https://192.168.49.2:6443/metrics: HTTPSConnectionPool(host='192.168.49.2', port=6443): Max retries exceeded with url: /metrics (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0xffff6b256f70>: Failed to establish a new connection: [Errno 111] Connection refused')), trying next possible URL.
2023-01-27 16:32:25 UTC | CORE | INFO | (pkg/collector/scheduler/scheduler.go:92 in Enter) | Scheduling check kube_apiserver_metrics with an interval of 15s
2023-01-27 16:32:25 UTC | CORE | INFO | (pkg/collector/scheduler/scheduler.go:92 in Enter) | Scheduling check container with an interval of 15s
2023-01-27 16:32:25 UTC | CORE | INFO | (pkg/collector/scheduler/scheduler.go:92 in Enter) | Scheduling check cpu with an interval of 15s
2023-01-27 16:32:25 UTC | CORE | INFO | (pkg/collector/scheduler/scheduler.go:92 in Enter) | Scheduling check disk with an interval of 15s
2023-01-27 16:32:25 UTC | CORE | INFO | (pkg/collector/scheduler/scheduler.go:92 in Enter) | Scheduling check docker with an interval of 15s
2023-01-27 16:32:25 UTC | CORE | INFO | (pkg/collector/scheduler/scheduler.go:92 in Enter) | Scheduling check file_handle with an interval of 15s
2023-01-27 16:32:25 UTC | CORE | INFO | (pkg/collector/scheduler/scheduler.go:92 in Enter) | Scheduling check io with an interval of 15s
2023-01-27 16:32:25 UTC | CORE | INFO | (pkg/collector/scheduler/scheduler.go:92 in Enter) | Scheduling check kubelet with an interval of 20s
2023-01-27 16:32:25 UTC | CORE | INFO | (pkg/collector/scheduler/scheduler.go:92 in Enter) | Scheduling check load with an interval of 15s
2023-01-27 16:32:25 UTC | CORE | INFO | (pkg/collector/scheduler/scheduler.go:92 in Enter) | Scheduling check memory with an interval of 15s
2023-01-27 16:32:25 UTC | CORE | INFO | (pkg/collector/scheduler/scheduler.go:92 in Enter) | Scheduling check network with an interval of 15s
2023-01-27 16:32:25 UTC | CORE | INFO | (pkg/util/cloudproviders/cloudproviders.go:56 in DetectCloudProvider) | No cloud provider detected
2023-01-27 16:32:26 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:38 in CheckStarted) | check:coredns | Running check...
2023-01-27 16:32:26 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:57 in CheckFinished) | check:coredns | Done running check
2023-01-27 16:32:26 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:38 in CheckStarted) | check:kubelet | Running check...
2023-01-27 16:32:26 UTC | CORE | INFO | (pkg/collector/scheduler/scheduler.go:92 in Enter) | Scheduling check ntp with an interval of 15m0s
2023-01-27 16:32:26 UTC | CORE | INFO | (pkg/collector/scheduler/scheduler.go:92 in Enter) | Scheduling check uptime with an interval of 15s
2023-01-27 16:32:26 UTC | CORE | INFO | (pkg/autodiscovery/autoconfig.go:204 in LoadAndRun) | Started config provider "file"
2023-01-27 16:32:26 UTC | CORE | INFO | (pkg/api/security/security.go:189 in getClusterAgentAuthToken) | Using configured cluster_agent.auth_token
2023-01-27 16:32:26 UTC | CORE | ERROR | (pkg/autodiscovery/config_poller.go:211 in collect) | Unable to collect configurations from provider endpoints-checks: temporary failure in clusterAgentClient, will retry later: "https://10.104.229.207:5005/version" is unavailable: Get "https://10.104.229.207:5005/version": dial tcp 10.104.229.207:5005: connect: connection refused
2023-01-27 16:32:26 UTC | CORE | INFO | (pkg/autodiscovery/autoconfig.go:202 in LoadAndRun) | Started config provider "endpoints-checks", polled every 10s
2023-01-27 16:32:26 UTC | CORE | INFO | (pkg/autodiscovery/autoconfig.go:204 in LoadAndRun) | Started config provider "kubernetes-container-allinone"
2023-01-27 16:32:26 UTC | CORE | ERROR | (pkg/autodiscovery/config_poller.go:211 in collect) | Unable to collect configurations from provider cluster-checks: temporary failure in clusterAgentClient, will retry later: try delay not elapsed yet
2023-01-27 16:32:26 UTC | CORE | INFO | (pkg/autodiscovery/autoconfig.go:202 in LoadAndRun) | Started config provider "cluster-checks", polled every 10s
2023-01-27 16:32:26 UTC | CORE | INFO | (pkg/util/kubernetes/clustername/clustername.go:75 in getClusterName) | Got cluster name mini from config
2023-01-27 16:32:26 UTC | CORE | INFO | (pkg/metadata/host/host_tags.go:101 in GetHostTags) | Adding both tags cluster_name and kube_cluster_name. You can use 'disable_cluster_name_tag_key' in the Agent config to keep the kube_cluster_name tag only
2023-01-27 16:32:26 UTC | CORE | WARN | (pkg/util/cloudproviders/gce/gce_tags.go:50 in getCachedTags) | unable to get tags from gce and cache is empty: GCE metadata API error: Get "http://169.254.169.254/computeMetadata/v1/?recursive=true": dial tcp 169.254.169.254:80: connect: connection refused
2023-01-27 16:32:26 UTC | CORE | INFO | (pkg/metadata/host/host_tags.go:130 in GetHostTags) | Unable to get host tags from source: gce - using cached host tags
2023-01-27 16:32:27 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:38 in CheckStarted) | check:ntp | Running check...
2023-01-27 16:32:27 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:57 in CheckFinished) | check:ntp | Done running check
2023-01-27 16:32:28 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:38 in CheckStarted) | check:uptime | Running check...
2023-01-27 16:32:28 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:57 in CheckFinished) | check:uptime | Done running check
2023-01-27 16:32:28 UTC | CORE | ERROR | (pkg/collector/worker/check_logger.go:69 in Error) | check:kubelet | Error running check: [{"message": "401 Client Error: Unauthorized for url: https://192.168.49.2:10250/metrics/cadvisor", "traceback": "Traceback (most recent call last):\n  File \"/opt/datadog-agent/embedded/lib/python3.8/site-packages/datadog_checks/base/checks/base.py\", line 1122, in run\n    self.check(instance)\n  File \"/opt/datadog-agent/embedded/lib/python3.8/site-packages/datadog_checks/kubelet/kubelet.py\", line 387, in check\n    self.process(self.cadvisor_scraper_config, metric_transformers=self.transformers)\n  File \"/opt/datadog-agent/embedded/lib/python3.8/site-packages/datadog_checks/base/checks/openmetrics/mixins.py\", line 573, in process\n    for metric in self.scrape_metrics(scraper_config):\n  File \"/opt/datadog-agent/embedded/lib/python3.8/site-packages/datadog_checks/base/checks/openmetrics/mixins.py\", line 500, in scrape_metrics\n    response = self.poll(scraper_config)\n  File \"/opt/datadog-agent/embedded/lib/python3.8/site-packages/datadog_checks/base/checks/openmetrics/mixins.py\", line 846, in poll\n    response.raise_for_status()\n  File \"/opt/datadog-agent/embedded/lib/python3.8/site-packages/requests/models.py\", line 1021, in raise_for_status\n    raise HTTPError(http_error_msg, response=self)\nrequests.exceptions.HTTPError: 401 Client Error: Unauthorized for url: https://192.168.49.2:10250/metrics/cadvisor\n"}]
2023-01-27 16:32:28 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:57 in CheckFinished) | check:kubelet | Done running check
2023-01-27 16:32:28 UTC | CORE | INFO | (pkg/api/security/security.go:189 in getClusterAgentAuthToken) | Using configured cluster_agent.auth_token
2023-01-27 16:32:29 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:38 in CheckStarted) | check:memory | Running check...
2023-01-27 16:32:29 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:57 in CheckFinished) | check:memory | Done running check
2023-01-27 16:32:30 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:38 in CheckStarted) | check:io | Running check...
2023-01-27 16:32:30 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:57 in CheckFinished) | check:io | Done running check
2023-01-27 16:32:31 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:38 in CheckStarted) | check:docker | Running check...
2023-01-27 16:32:31 UTC | CORE | INFO | (pkg/util/containers/metrics/provider/provider.go:210 in updateEffectiveCollectors) | Using metrics collector: docker for runtime: docker
2023-01-27 16:32:31 UTC | CORE | INFO | (pkg/util/containers/metrics/provider/provider.go:210 in updateEffectiveCollectors) | Using metrics collector: kubelet for runtime: cri-o
2023-01-27 16:32:31 UTC | CORE | INFO | (pkg/util/containers/metrics/provider/provider.go:210 in updateEffectiveCollectors) | Using metrics collector: kubelet for runtime: containerd
2023-01-27 16:32:31 UTC | CORE | WARN | (pkg/util/cgroups/pid_mapper.go:98 in getPidMapper) | Usage of cgroupv2 detected but the Agent does not seem to run in host cgroup namespace. Make sure to run with --cgroupns=host, some feature may not work otherwise
2023-01-27 16:32:31 UTC | CORE | INFO | (pkg/util/containers/metrics/provider/provider.go:213 in updateEffectiveCollectors) | Replaced old collector: docker by new collector: system for runtime: docker
2023-01-27 16:32:31 UTC | CORE | INFO | (pkg/util/containers/metrics/provider/provider.go:213 in updateEffectiveCollectors) | Replaced old collector: kubelet by new collector: system for runtime: containerd
2023-01-27 16:32:31 UTC | CORE | INFO | (pkg/util/containers/metrics/provider/provider.go:213 in updateEffectiveCollectors) | Replaced old collector: kubelet by new collector: system for runtime: cri-o
2023-01-27 16:32:31 UTC | CORE | INFO | (pkg/util/containers/metrics/provider/provider.go:210 in updateEffectiveCollectors) | Using metrics collector: system for runtime: garden
2023-01-27 16:32:31 UTC | CORE | INFO | (pkg/util/containers/metrics/provider/provider.go:210 in updateEffectiveCollectors) | Using metrics collector: system for runtime: podman
2023-01-27 16:32:31 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:57 in CheckFinished) | check:docker | Done running check
2023-01-27 16:32:32 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:38 in CheckStarted) | check:cpu | Running check...
2023-01-27 16:32:32 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:57 in CheckFinished) | check:cpu | Done running check
2023-01-27 16:32:32 UTC | CORE | INFO | (pkg/api/security/security.go:189 in getClusterAgentAuthToken) | Using configured cluster_agent.auth_token
2023-01-27 16:32:33 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:38 in CheckStarted) | check:kube_apiserver_metrics | Running check...
2023-01-27 16:32:33 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:57 in CheckFinished) | check:kube_apiserver_metrics | Done running check
2023-01-27 16:32:35 UTC | CORE | INFO | (pkg/metadata/host/host_tags.go:130 in GetHostTags) | Unable to get host tags from source: kubernetes - using cached host tags
2023-01-27 16:32:36 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:38 in CheckStarted) | check:network | Running check...
2023-01-27 16:32:36 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:57 in CheckFinished) | check:network | Done running check
2023-01-27 16:32:36 UTC | CORE | ERROR | (pkg/autodiscovery/config_poller.go:211 in collect) | Unable to collect configurations from provider endpoints-checks: temporary failure in clusterAgentClient, will retry later: try delay not elapsed yet
2023-01-27 16:32:36 UTC | CORE | ERROR | (pkg/autodiscovery/config_poller.go:149 in poll) | Cache processing of cluster-checks configuration provider failed: temporary failure in clusterAgentClient, will retry later: try delay not elapsed yet
2023-01-27 16:32:36 UTC | CORE | INFO | (pkg/metadata/host/host.go:164 in getNetworkMeta) | could not get network metadata: could not detect network ID
2023-01-27 16:32:36 UTC | CORE | INFO | (pkg/serializer/serializer.go:403 in sendMetadata) | Sent metadata payload, size (raw/compressed): 5505/2204 bytes.
2023-01-27 16:32:37 UTC | CORE | INFO | (pkg/forwarder/transaction/transaction.go:378 in internalProcess) | Successfully posted payload to "https://7-41-1-app.agent.datadoghq.com/intake/?api_key=***************************08bfc", the agent will only log transaction success every 500 transactions
2023-01-27 16:32:37 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:38 in CheckStarted) | check:load | Running check...
2023-01-27 16:32:37 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:57 in CheckFinished) | check:load | Done running check
2023-01-27 16:32:38 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:38 in CheckStarted) | check:file_handle | Running check...
2023-01-27 16:32:38 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:57 in CheckFinished) | check:file_handle | Done running check
2023-01-27 16:32:39 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:38 in CheckStarted) | check:disk | Running check...
2023-01-27 16:32:39 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:57 in CheckFinished) | check:disk | Done running check
2023-01-27 16:32:40 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:38 in CheckStarted) | check:container | Running check...
2023-01-27 16:32:40 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:57 in CheckFinished) | check:container | Done running check
2023-01-27 16:32:41 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:38 in CheckStarted) | check:coredns | Running check...
2023-01-27 16:32:41 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:57 in CheckFinished) | check:coredns | Done running check
2023-01-27 16:32:43 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:38 in CheckStarted) | check:uptime | Running check...
2023-01-27 16:32:43 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:57 in CheckFinished) | check:uptime | Done running check
2023-01-27 16:32:44 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:38 in CheckStarted) | check:memory | Running check...
2023-01-27 16:32:44 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:57 in CheckFinished) | check:memory | Done running check
2023-01-27 16:32:45 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:38 in CheckStarted) | check:io | Running check...
2023-01-27 16:32:45 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:57 in CheckFinished) | check:io | Done running check
2023-01-27 16:32:46 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:38 in CheckStarted) | check:docker | Running check...
2023-01-27 16:32:46 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:57 in CheckFinished) | check:docker | Done running check
2023-01-27 16:32:46 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:38 in CheckStarted) | check:kubelet | Running check...
2023-01-27 16:32:46 UTC | CORE | INFO | (pkg/api/security/security.go:189 in getClusterAgentAuthToken) | Using configured cluster_agent.auth_token
2023-01-27 16:32:46 UTC | CORE | INFO | (pkg/api/security/security.go:189 in getClusterAgentAuthToken) | Using configured cluster_agent.auth_token
2023-01-27 16:32:46 UTC | CORE | ERROR | (pkg/autodiscovery/config_poller.go:149 in poll) | Cache processing of cluster-checks configuration provider failed: temporary failure in clusterAgentClient, will retry later: "https://10.104.229.207:5005/version" is unavailable: Get "https://10.104.229.207:5005/version": dial tcp 10.104.229.207:5005: connect: connection refused
2023-01-27 16:32:46 UTC | CORE | ERROR | (pkg/autodiscovery/config_poller.go:211 in collect) | Unable to collect configurations from provider endpoints-checks: temporary failure in clusterAgentClient, will retry later: "https://10.104.229.207:5005/version" is unavailable: Get "https://10.104.229.207:5005/version": dial tcp 10.104.229.207:5005: connect: connection refused
2023-01-27 16:32:47 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:38 in CheckStarted) | check:cpu | Running check...
2023-01-27 16:32:47 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:57 in CheckFinished) | check:cpu | Done running check
2023-01-27 16:32:48 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:38 in CheckStarted) | check:kube_apiserver_metrics | Running check...
2023-01-27 16:32:48 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:57 in CheckFinished) | check:kube_apiserver_metrics | Done running check
2023-01-27 16:32:50 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:57 in CheckFinished) | check:kubelet | Done running check
2023-01-27 16:32:51 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:38 in CheckStarted) | check:network | Running check...
2023-01-27 16:32:51 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:57 in CheckFinished) | check:network | Done running check
2023-01-27 16:32:52 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:38 in CheckStarted) | check:load | Running check...
2023-01-27 16:32:52 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:57 in CheckFinished) | check:load | Done running check
2023-01-27 16:32:53 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:38 in CheckStarted) | check:file_handle | Running check...
2023-01-27 16:32:53 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:57 in CheckFinished) | check:file_handle | Done running check
2023-01-27 16:32:54 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:38 in CheckStarted) | check:disk | Running check...
2023-01-27 16:32:54 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:57 in CheckFinished) | check:disk | Done running check
2023-01-27 16:32:54 UTC | CORE | ERROR | (pkg/workloadmeta/collectors/internal/kubemetadata/kubemetadata.go:73 in Start) | Could not initialise the communication with the cluster agent: temporary failure in clusterAgentClient, will retry later: try delay not elapsed yet
2023-01-27 16:32:55 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:38 in CheckStarted) | check:container | Running check...
2023-01-27 16:32:55 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:57 in CheckFinished) | check:container | Done running check
2023-01-27 16:32:55 UTC | CORE | INFO | (pkg/collector/scheduler/scheduler.go:92 in Enter) | Scheduling check datadog_cluster_agent with an interval of 15s
2023-01-27 16:32:56 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:38 in CheckStarted) | check:coredns | Running check...
2023-01-27 16:32:56 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:57 in CheckFinished) | check:coredns | Done running check
2023-01-27 16:32:56 UTC | CORE | ERROR | (pkg/autodiscovery/config_poller.go:149 in poll) | Cache processing of cluster-checks configuration provider failed: temporary failure in clusterAgentClient, will retry later: try delay not elapsed yet
2023-01-27 16:32:56 UTC | CORE | ERROR | (pkg/autodiscovery/config_poller.go:211 in collect) | Unable to collect configurations from provider endpoints-checks: temporary failure in clusterAgentClient, will retry later: try delay not elapsed yet
2023-01-27 16:32:58 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:38 in CheckStarted) | check:uptime | Running check...
2023-01-27 16:32:58 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:57 in CheckFinished) | check:uptime | Done running check
2023-01-27 16:32:59 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:38 in CheckStarted) | check:memory | Running check...
2023-01-27 16:32:59 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:57 in CheckFinished) | check:memory | Done running check
2023-01-27 16:33:00 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:38 in CheckStarted) | check:io | Running check...
2023-01-27 16:33:00 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:57 in CheckFinished) | check:io | Done running check
2023-01-27 16:33:01 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:38 in CheckStarted) | check:docker | Running check...
2023-01-27 16:33:01 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:57 in CheckFinished) | check:docker | Done running check
2023-01-27 16:33:02 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:38 in CheckStarted) | check:cpu | Running check...
2023-01-27 16:33:02 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:57 in CheckFinished) | check:cpu | Done running check
2023-01-27 16:33:03 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:38 in CheckStarted) | check:kube_apiserver_metrics | Running check...
2023-01-27 16:33:03 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:57 in CheckFinished) | check:kube_apiserver_metrics | Done running check
2023-01-27 16:33:05 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:38 in CheckStarted) | check:datadog_cluster_agent | Running check...
2023-01-27 16:33:05 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:57 in CheckFinished) | check:datadog_cluster_agent | Done running check
2023-01-27 16:33:06 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:38 in CheckStarted) | check:network | Running check...
2023-01-27 16:33:06 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:57 in CheckFinished) | check:network | Done running check
2023-01-27 16:33:06 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:38 in CheckStarted) | check:kubelet | Running check...
2023-01-27 16:33:06 UTC | CORE | ERROR | (pkg/autodiscovery/config_poller.go:211 in collect) | Unable to collect configurations from provider endpoints-checks: temporary failure in clusterAgentClient, will retry later: try delay not elapsed yet
2023-01-27 16:33:06 UTC | CORE | ERROR | (pkg/autodiscovery/config_poller.go:149 in poll) | Cache processing of cluster-checks configuration provider failed: temporary failure in clusterAgentClient, will retry later: try delay not elapsed yet
2023-01-27 16:33:07 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:38 in CheckStarted) | check:load | Running check...
2023-01-27 16:33:07 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:57 in CheckFinished) | check:load | Done running check
2023-01-27 16:33:08 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:38 in CheckStarted) | check:file_handle | Running check...
2023-01-27 16:33:08 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:57 in CheckFinished) | check:file_handle | Done running check
2023-01-27 16:33:09 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:38 in CheckStarted) | check:disk | Running check...
2023-01-27 16:33:09 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:57 in CheckFinished) | check:disk | Done running check
2023-01-27 16:33:10 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:38 in CheckStarted) | check:container | Running check...
2023-01-27 16:33:10 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:57 in CheckFinished) | check:container | Done running check
2023-01-27 16:33:10 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:57 in CheckFinished) | check:kubelet | Done running check
2023-01-27 16:33:11 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:38 in CheckStarted) | check:coredns | Running check...
2023-01-27 16:33:11 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:57 in CheckFinished) | check:coredns | Done running check
2023-01-27 16:33:13 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:38 in CheckStarted) | check:uptime | Running check...
2023-01-27 16:33:13 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:57 in CheckFinished) | check:uptime | Done running check
2023-01-27 16:33:14 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:38 in CheckStarted) | check:memory | Running check...
2023-01-27 16:33:14 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:57 in CheckFinished) | check:memory | Done running check
2023-01-27 16:33:15 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:38 in CheckStarted) | check:io | Running check...
2023-01-27 16:33:15 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:57 in CheckFinished) | check:io | Done running check
2023-01-27 16:33:16 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:38 in CheckStarted) | check:docker | Running check...
2023-01-27 16:33:16 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:57 in CheckFinished) | check:docker | Done running check
2023-01-27 16:33:16 UTC | CORE | ERROR | (pkg/autodiscovery/config_poller.go:211 in collect) | Unable to collect configurations from provider endpoints-checks: temporary failure in clusterAgentClient, will retry later: try delay not elapsed yet
2023-01-27 16:33:16 UTC | CORE | ERROR | (pkg/autodiscovery/config_poller.go:149 in poll) | Cache processing of cluster-checks configuration provider failed: temporary failure in clusterAgentClient, will retry later: try delay not elapsed yet
2023-01-27 16:33:17 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:38 in CheckStarted) | check:cpu | Running check...
2023-01-27 16:33:17 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:57 in CheckFinished) | check:cpu | Done running check
2023-01-27 16:33:18 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:38 in CheckStarted) | check:kube_apiserver_metrics | Running check...
2023-01-27 16:33:18 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:57 in CheckFinished) | check:kube_apiserver_metrics | Done running check
2023-01-27 16:33:20 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:38 in CheckStarted) | check:datadog_cluster_agent | Running check...
2023-01-27 16:33:20 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:57 in CheckFinished) | check:datadog_cluster_agent | Done running check
2023-01-27 16:33:21 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:38 in CheckStarted) | check:network | Running check...
2023-01-27 16:33:21 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:57 in CheckFinished) | check:network | Done running check
2023-01-27 16:33:22 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:38 in CheckStarted) | check:load | Running check...
2023-01-27 16:33:22 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:57 in CheckFinished) | check:load | Done running check
2023-01-27 16:33:23 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:38 in CheckStarted) | check:file_handle | Running check...
2023-01-27 16:33:23 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:57 in CheckFinished) | check:file_handle | Done running check
2023-01-27 16:33:24 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:38 in CheckStarted) | check:disk | Running check...
2023-01-27 16:33:24 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:57 in CheckFinished) | check:disk | Done running check
2023-01-27 16:33:24 UTC | CORE | INFO | (pkg/api/security/security.go:189 in getClusterAgentAuthToken) | Using configured cluster_agent.auth_token
2023-01-27 16:33:24 UTC | CORE | INFO | (pkg/util/clusteragent/clusteragent.go:130 in init) | Successfully connected to the Datadog Cluster Agent 7.41.1+commit.4f39b9e
2023-01-27 16:33:24 UTC | CORE | INFO | (pkg/workloadmeta/store.go:337 in startCandidates) | workloadmeta collector "kube_metadata" started successfully
2023-01-27 16:33:25 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:38 in CheckStarted) | check:container | Running check...
2023-01-27 16:33:25 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:57 in CheckFinished) | check:container | Done running check
2023-01-27 16:33:26 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:38 in CheckStarted) | check:coredns | Running check...
2023-01-27 16:33:26 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:57 in CheckFinished) | check:coredns | Done running check, next runs will be logged every 500 runs
2023-01-27 16:33:26 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:38 in CheckStarted) | check:kubelet | Running check...
2023-01-27 16:33:28 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:38 in CheckStarted) | check:uptime | Running check...
2023-01-27 16:33:28 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:57 in CheckFinished) | check:uptime | Done running check, next runs will be logged every 500 runs
2023-01-27 16:33:29 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:38 in CheckStarted) | check:memory | Running check...
2023-01-27 16:33:29 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:57 in CheckFinished) | check:memory | Done running check, next runs will be logged every 500 runs
2023-01-27 16:33:30 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:38 in CheckStarted) | check:io | Running check...
2023-01-27 16:33:30 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:57 in CheckFinished) | check:io | Done running check, next runs will be logged every 500 runs
2023-01-27 16:33:30 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:57 in CheckFinished) | check:kubelet | Done running check
2023-01-27 16:33:31 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:38 in CheckStarted) | check:docker | Running check...
2023-01-27 16:33:31 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:57 in CheckFinished) | check:docker | Done running check, next runs will be logged every 500 runs
2023-01-27 16:33:32 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:38 in CheckStarted) | check:cpu | Running check...
2023-01-27 16:33:32 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:57 in CheckFinished) | check:cpu | Done running check, next runs will be logged every 500 runs
2023-01-27 16:33:33 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:38 in CheckStarted) | check:kube_apiserver_metrics | Running check...
2023-01-27 16:33:33 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:57 in CheckFinished) | check:kube_apiserver_metrics | Done running check, next runs will be logged every 500 runs
2023-01-27 16:33:35 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:38 in CheckStarted) | check:datadog_cluster_agent | Running check...
2023-01-27 16:33:35 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:57 in CheckFinished) | check:datadog_cluster_agent | Done running check
2023-01-27 16:33:36 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:38 in CheckStarted) | check:network | Running check...
2023-01-27 16:33:36 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:57 in CheckFinished) | check:network | Done running check, next runs will be logged every 500 runs
2023-01-27 16:33:37 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:38 in CheckStarted) | check:load | Running check...
2023-01-27 16:33:37 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:57 in CheckFinished) | check:load | Done running check, next runs will be logged every 500 runs
2023-01-27 16:33:38 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:38 in CheckStarted) | check:file_handle | Running check...
2023-01-27 16:33:38 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:57 in CheckFinished) | check:file_handle | Done running check, next runs will be logged every 500 runs
2023-01-27 16:33:39 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:38 in CheckStarted) | check:disk | Running check...
2023-01-27 16:33:39 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:57 in CheckFinished) | check:disk | Done running check, next runs will be logged every 500 runs
2023-01-27 16:33:39 UTC | CORE | INFO | (pkg/collector/scheduler/scheduler.go:131 in Cancel) | Unscheduling check coredns:a6d778794982e186
2023-01-27 16:33:39 UTC | CORE | INFO | (pkg/collector/scheduler/scheduler.go:92 in Enter) | Scheduling check coredns with an interval of 15s
2023-01-27 16:33:39 UTC | CORE | INFO | (pkg/collector/scheduler/scheduler.go:131 in Cancel) | Unscheduling check datadog_cluster_agent:7ad1c752791e7001
2023-01-27 16:33:39 UTC | CORE | INFO | (pkg/collector/scheduler/scheduler.go:92 in Enter) | Scheduling check datadog_cluster_agent with an interval of 15s
2023-01-27 16:33:40 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:38 in CheckStarted) | check:container | Running check...
2023-01-27 16:33:40 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:57 in CheckFinished) | check:container | Done running check, next runs will be logged every 500 runs
2023-01-27 16:33:42 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:38 in CheckStarted) | check:coredns | Running check...
2023-01-27 16:33:42 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:57 in CheckFinished) | check:coredns | Done running check
2023-01-27 16:33:46 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:38 in CheckStarted) | check:kubelet | Running check...
2023-01-27 16:33:49 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:38 in CheckStarted) | check:datadog_cluster_agent | Running check...
2023-01-27 16:33:49 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:57 in CheckFinished) | check:datadog_cluster_agent | Done running check
2023-01-27 16:33:50 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:57 in CheckFinished) | check:kubelet | Done running check, next runs will be logged every 500 runs
2023-01-27 16:33:57 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:38 in CheckStarted) | check:coredns | Running check...
2023-01-27 16:33:57 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:57 in CheckFinished) | check:coredns | Done running check
2023-01-27 16:34:04 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:38 in CheckStarted) | check:datadog_cluster_agent | Running check...
2023-01-27 16:34:04 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:57 in CheckFinished) | check:datadog_cluster_agent | Done running check
2023-01-27 16:34:12 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:38 in CheckStarted) | check:coredns | Running check...
2023-01-27 16:34:12 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:57 in CheckFinished) | check:coredns | Done running check
2023-01-27 16:34:19 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:38 in CheckStarted) | check:datadog_cluster_agent | Running check...
2023-01-27 16:34:19 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:57 in CheckFinished) | check:datadog_cluster_agent | Done running check
2023-01-27 16:34:27 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:38 in CheckStarted) | check:coredns | Running check...
2023-01-27 16:34:27 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:57 in CheckFinished) | check:coredns | Done running check
2023-01-27 16:34:34 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:38 in CheckStarted) | check:datadog_cluster_agent | Running check...
2023-01-27 16:34:34 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:57 in CheckFinished) | check:datadog_cluster_agent | Done running check
2023-01-27 16:34:34 UTC | CORE | INFO | (pkg/autodiscovery/config_poller.go:105 in stream) | kubernetes-container-allinone provider: collected 1 new configurations, removed 0
2023-01-27 16:34:42 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:38 in CheckStarted) | check:coredns | Running check...
2023-01-27 16:34:42 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:57 in CheckFinished) | check:coredns | Done running check, next runs will be logged every 500 runs
2023-01-27 16:34:49 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:38 in CheckStarted) | check:datadog_cluster_agent | Running check...
2023-01-27 16:34:49 UTC | CORE | INFO | (pkg/collector/worker/check_logger.go:57 in CheckFinished) | check:datadog_cluster_agent | Done running check, next runs will be logged every 500 runs
2023-01-27 16:35:37 UTC | CORE | INFO | (cmd/agent/api/internal/agent/agent.go:181 in getStatus) | Got a request for the status. Making status.
2023-01-27 16:35:37 UTC | CORE | INFO | (pkg/metadata/inventories/host_metadata.go:138 in getHostMetadata) | cloud provider not found in agent metadata cache
2023-01-27 16:37:15 UTC | CORE | INFO | (cmd/agent/api/internal/agent/agent.go:181 in getStatus) | Got a request for the status. Making status.
2023-01-27 16:37:22 UTC | CORE | INFO | (pkg/serializer/serializer.go:403 in sendMetadata) | Sent metadata payload, size (raw/compressed): 2632/901 bytes.
2023-01-27 16:37:36 UTC | CORE | INFO | (pkg/serializer/serializer.go:427 in SendProcessesMetadata) | Sent processes metadata payload, size: 1339 bytes.
2023-01-27 16:37:51 UTC | CORE | INFO | (pkg/config/settings/http/server.go:136 in setConfigValue) | Got a request to change a setting: log_level
2023-01-27 16:37:52 UTC | CORE | DEBUG | (pkg/collector/worker/check_logger.go:42 in CheckStarted) | check:load | Running check...
2023-01-27 16:37:52 UTC | CORE | DEBUG | (pkg/collector/worker/check_logger.go:59 in CheckFinished) | check:load | Done running check
2023-01-27 16:37:53 UTC | CORE | DEBUG | (pkg/collector/worker/check_logger.go:42 in CheckStarted) | check:file_handle | Running check...
2023-01-27 16:37:53 UTC | CORE | DEBUG | (pkg/collector/worker/check_logger.go:59 in CheckFinished) | check:file_handle | Done running check
2023-01-27 16:37:54 UTC | CORE | DEBUG | (pkg/collector/worker/check_logger.go:42 in CheckStarted) | check:disk | Running check...
2023-01-27 16:37:54 UTC | CORE | DEBUG | (pkg/collector/python/check.go:84 in runCheck) | Running python check disk (version: '4.7.1', id: 'disk:e5dffb8bef24336f')
2023-01-27 16:37:54 UTC | CORE | DEBUG | (pkg/collector/worker/check_logger.go:59 in CheckFinished) | check:disk | Done running check
2023-01-27 16:37:54 UTC | CORE | DEBUG | (pkg/config/config.go:1827 in IsCloudProviderEnabled) | cloud_provider_metadata is set to [aws gcp azure alibaba oracle ibm] in agent configuration, trying endpoints for AWS Cloud Provider
2023-01-27 16:37:54 UTC | CORE | DEBUG | (pkg/config/config.go:1827 in IsCloudProviderEnabled) | cloud_provider_metadata is set to [aws gcp azure alibaba oracle ibm] in agent configuration, trying endpoints for AWS Cloud Provider
2023-01-27 16:37:54 UTC | CORE | DEBUG | (pkg/config/config.go:1827 in IsCloudProviderEnabled) | cloud_provider_metadata is set to [aws gcp azure alibaba oracle ibm] in agent configuration, trying endpoints for AWS Cloud Provider
2023-01-27 16:37:54 UTC | CORE | DEBUG | (pkg/util/ecs/metadata/clients.go:65 in V1) | ECS metadata v1 client init error: temporary failure in ecsutil-meta-v1, will retry later: try delay not elapsed yet
2023-01-27 16:37:54 UTC | CORE | DEBUG | (pkg/workloadmeta/store.go:331 in startCandidates) | workloadmeta collector "ecs" could not start, but will retry. error: temporary failure in ecsutil-meta-v1, will retry later: try delay not elapsed yet
2023-01-27 16:37:54 UTC | CORE | DEBUG | (pkg/util/kubernetes/kubelet/podwatcher.go:147 in computeChanges) | Found 0 changed pods out of 11
2023-01-27 16:37:55 UTC | CORE | DEBUG | (pkg/aggregator/demultiplexer.go:144 in sendIterableSeries) | Demultiplexer: sendIterableSeries: start sending iterable series to the serializer
2023-01-27 16:37:55 UTC | CORE | DEBUG | (pkg/aggregator/aggregator.go:604 in sendServiceChecks) | Flushing 24 service checks to the forwarder
2023-01-27 16:37:55 UTC | CORE | DEBUG | (pkg/aggregator/demultiplexer.go:150 in sendIterableSeries) | Demultiplexer: sendIterableSeries: stop routine
2023-01-27 16:37:55 UTC | CORE | DEBUG | (pkg/logs/internal/launchers/file/launcher.go:141 in scan) | Scan - got 0 files from FilesToTail and currently tailing 0 files
2023-01-27 16:37:55 UTC | CORE | DEBUG | (pkg/logs/internal/launchers/file/launcher.go:195 in scan) | After stopping tailers, there are 0 tailers running.
2023-01-27 16:37:55 UTC | CORE | DEBUG | (pkg/logs/internal/launchers/file/launcher.go:212 in scan) | After starting new tailers, there are 0 tailers running. Limit is 500.
2023-01-27 16:37:55 UTC | CORE | DEBUG | (pkg/collector/worker/check_logger.go:42 in CheckStarted) | check:container | Running check...
2023-01-27 16:37:55 UTC | CORE | DEBUG | (pkg/collector/corechecks/containers/generic/processor.go:116 in Run) | OpenFiles count for: &{{container 59f8054dd8439dda298171fecb121928e077dd0e81835fdfbbd86dd47be7e925} {agent  map[] map[annotation.io.kubernetes.container.hash:b7d4b3a9 annotation.io.kubernetes.container.ports:[{"name":"dogstatsdport","containerPort":8125,"protocol":"UDP"}] annotation.io.kubernetes.container.restartCount:0 annotation.io.kubernetes.container.terminationMessagePath:/dev/termination-log annotation.io.kubernetes.container.terminationMessagePolicy:File annotation.io.kubernetes.pod.terminationGracePeriod:30 io.kubernetes.container.logpath:/var/log/pods/default_datadog-lzcr8_3532449e-51ba-4864-a468-d7a4da5b094f/agent/0.log io.kubernetes.container.name:agent io.kubernetes.docker.type:container io.kubernetes.pod.name:datadog-lzcr8 io.kubernetes.pod.namespace:default io.kubernetes.pod.uid:3532449e-51ba-4864-a468-d7a4da5b094f io.kubernetes.sandbox.id:76d45bcbf4a6c61636d13d5c4c30ac05dcb850055dc7eed90a54e246fe6695bc maintainer:Datadog <package@datadoghq.com>]} map[CURL_CA_BUNDLE:/opt/datadog-agent/embedded/ssl/certs/cacert.pem DATADOG_CLUSTER_AGENT_ADMISSION_CONTROLLER_PORT:tcp://10.98.222.107:443 DATADOG_CLUSTER_AGENT_ADMISSION_CONTROLLER_PORT_443_TCP:tcp://10.98.222.107:443 DATADOG_CLUSTER_AGENT_ADMISSION_CONTROLLER_PORT_443_TCP_ADDR:10.98.222.107 DATADOG_CLUSTER_AGENT_ADMISSION_CONTROLLER_PORT_443_TCP_PORT:443 DATADOG_CLUSTER_AGENT_ADMISSION_CONTROLLER_PORT_443_TCP_PROTO:tcp DATADOG_CLUSTER_AGENT_ADMISSION_CONTROLLER_SERVICE_HOST:10.98.222.107 DATADOG_CLUSTER_AGENT_ADMISSION_CONTROLLER_SERVICE_PORT:443 DATADOG_CLUSTER_AGENT_PORT:tcp://10.104.229.207:5005 DATADOG_CLUSTER_AGENT_PORT_5005_TCP:tcp://10.104.229.207:5005 DATADOG_CLUSTER_AGENT_PORT_5005_TCP_ADDR:10.104.229.207 DATADOG_CLUSTER_AGENT_PORT_5005_TCP_PORT:5005 DATADOG_CLUSTER_AGENT_PORT_5005_TCP_PROTO:tcp DATADOG_CLUSTER_AGENT_SERVICE_HOST:10.104.229.207 DATADOG_CLUSTER_AGENT_SERVICE_PORT:5005 DATADOG_CLUSTER_AGENT_SERVICE_PORT_AGENTPORT:5005 DATADOG_PORT:udp://10.96.246.183:8125 DATADOG_PORT_8125_UDP:udp://10.96.246.183:8125 DATADOG_PORT_8125_UDP_ADDR:10.96.246.183 DATADOG_PORT_8125_UDP_PORT:8125 DATADOG_PORT_8125_UDP_PROTO:udp DATADOG_SERVICE_HOST:10.96.246.183 DATADOG_SERVICE_PORT:8125 DATADOG_SERVICE_PORT_DOGSTATSD:8125 DD_API_KEY:"***************************08bfc" DD_APM_ENABLED:false DD_CLUSTER_AGENT_AUTH_TOKEN:4tcpr3fNUzQmp1frQEqVx8EOLe8LN2fP DD_CLUSTER_AGENT_ENABLED:true DD_CLUSTER_AGENT_KUBERNETES_SERVICE_NAME:datadog-cluster-agent DD_CLUSTER_NAME:mini DD_DOGSTATSD_NON_LOCAL_TRAFFIC:true DD_DOGSTATSD_PORT:8125 DD_DOGSTATSD_SOCKET:/var/run/datadog/dsd.socket DD_EXPVAR_PORT:6000 DD_EXTRA_CONFIG_PROVIDERS:clusterchecks endpointschecks DD_HEALTH_PORT:5555 DD_IGNORE_AUTOCONF:etcd kube_controller_manager kube_scheduler kubernetes_state DD_KUBELET_TLS_VERIFY:false DD_KUBERNETES_KUBELET_HOST:192.168.49.2 DD_LOGS_CONFIG_AUTO_MULTI_LINE_DETECTION:false DD_LOGS_CONFIG_CONTAINER_COLLECT_ALL:false DD_LOGS_CONFIG_K8S_CONTAINER_USE_FILE:true DD_LOGS_ENABLED:true DD_LOG_LEVEL:INFO DD_PYTHON_VERSION:3 DD_SECRET_BACKEND_ARGUMENTS:/etc/secret-volume DD_SECRET_BACKEND_COMMAND:/readsecret_multiple_providers.sh DD_SECRET_BACKEND_COMMAND_ALLOW_GROUP_EXEC_PERM:true DEBIAN_FRONTEND:noninteractive DOCKER_DD_AGENT:true GODEBUG:x509ignoreCN=0 KUBERNETES:yes KUBERNETES_PORT:tcp://10.96.0.1:443 KUBERNETES_PORT_443_TCP:tcp://10.96.0.1:443 KUBERNETES_PORT_443_TCP_ADDR:10.96.0.1 KUBERNETES_PORT_443_TCP_PORT:443 KUBERNETES_PORT_443_TCP_PROTO:tcp KUBERNETES_SERVICE_HOST:10.96.0.1 KUBERNETES_SERVICE_PORT:443 KUBERNETES_SERVICE_PORT_HTTPS:443 LD_PRELOAD:/lib/x86_64-linux-gnu/nosys.so PATH:/opt/datadog-agent/bin/agent/:/opt/datadog-agent/embedded/bin/:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin S6_BEHAVIOUR_IF_STAGE2_FAILS:2 S6_KEEP_ENV:1 S6_LOGGING:0 S6_READ_ONLY_ROOT:1] datadog-lzcr8 {docker-pullable:********@sha256:4992b38387ca6dbb732f3493ed66cdf5efe7e08154b7e9e4159b1735dbaabe3f gcr.io/datadoghq/agent:7.41.1 gcr.io/datadoghq/agent agent 7.41.1} map[] 3317261 [{dogstatsdport 8125 UDP} { 8125 udp} { 8126 tcp}] docker {true running unknown 2023-01-27 16:32:21.734951171 +0000 UTC 2023-01-27 16:32:21.838310713 +0000 UTC 0001-01-01 00:00:00 +0000 UTC <nil>} []} not available through collector "system", err: unable to read any PID open FDs for cgroup id: 59f8054dd8439dda298171fecb121928e077dd0e81835fdfbbd86dd47be7e925. Unable to get count of open files
2023-01-27 16:37:55 UTC | CORE | DEBUG | (pkg/collector/corechecks/containers/generic/processor_network.go:50 in Process) | Gathering network metrics for container: &{{container 59f8054dd8439dda298171fecb121928e077dd0e81835fdfbbd86dd47be7e925} {agent  map[] map[annotation.io.kubernetes.container.hash:b7d4b3a9 annotation.io.kubernetes.container.ports:[{"name":"dogstatsdport","containerPort":8125,"protocol":"UDP"}] annotation.io.kubernetes.container.restartCount:0 annotation.io.kubernetes.container.terminationMessagePath:/dev/termination-log annotation.io.kubernetes.container.terminationMessagePolicy:File annotation.io.kubernetes.pod.terminationGracePeriod:30 io.kubernetes.container.logpath:/var/log/pods/default_datadog-lzcr8_3532449e-51ba-4864-a468-d7a4da5b094f/agent/0.log io.kubernetes.container.name:agent io.kubernetes.docker.type:container io.kubernetes.pod.name:datadog-lzcr8 io.kubernetes.pod.namespace:default io.kubernetes.pod.uid:3532449e-51ba-4864-a468-d7a4da5b094f io.kubernetes.sandbox.id:76d45bcbf4a6c61636d13d5c4c30ac05dcb850055dc7eed90a54e246fe6695bc maintainer:Datadog <package@datadoghq.com>]} map[CURL_CA_BUNDLE:/opt/datadog-agent/embedded/ssl/certs/cacert.pem DATADOG_CLUSTER_AGENT_ADMISSION_CONTROLLER_PORT:tcp://10.98.222.107:443 DATADOG_CLUSTER_AGENT_ADMISSION_CONTROLLER_PORT_443_TCP:tcp://10.98.222.107:443 DATADOG_CLUSTER_AGENT_ADMISSION_CONTROLLER_PORT_443_TCP_ADDR:10.98.222.107 DATADOG_CLUSTER_AGENT_ADMISSION_CONTROLLER_PORT_443_TCP_PORT:443 DATADOG_CLUSTER_AGENT_ADMISSION_CONTROLLER_PORT_443_TCP_PROTO:tcp DATADOG_CLUSTER_AGENT_ADMISSION_CONTROLLER_SERVICE_HOST:10.98.222.107 DATADOG_CLUSTER_AGENT_ADMISSION_CONTROLLER_SERVICE_PORT:443 DATADOG_CLUSTER_AGENT_PORT:tcp://10.104.229.207:5005 DATADOG_CLUSTER_AGENT_PORT_5005_TCP:tcp://10.104.229.207:5005 DATADOG_CLUSTER_AGENT_PORT_5005_TCP_ADDR:10.104.229.207 DATADOG_CLUSTER_AGENT_PORT_5005_TCP_PORT:5005 DATADOG_CLUSTER_AGENT_PORT_5005_TCP_PROTO:tcp DATADOG_CLUSTER_AGENT_SERVICE_HOST:10.104.229.207 DATADOG_CLUSTER_AGENT_SERVICE_PORT:5005 DATADOG_CLUSTER_AGENT_SERVICE_PORT_AGENTPORT:5005 DATADOG_PORT:udp://10.96.246.183:8125 DATADOG_PORT_8125_UDP:udp://10.96.246.183:8125 DATADOG_PORT_8125_UDP_ADDR:10.96.246.183 DATADOG_PORT_8125_UDP_PORT:8125 DATADOG_PORT_8125_UDP_PROTO:udp DATADOG_SERVICE_HOST:10.96.246.183 DATADOG_SERVICE_PORT:8125 DATADOG_SERVICE_PORT_DOGSTATSD:8125 DD_API_KEY:"***************************08bfc" DD_APM_ENABLED:false DD_CLUSTER_AGENT_AUTH_TOKEN:4tcpr3fNUzQmp1frQEqVx8EOLe8LN2fP DD_CLUSTER_AGENT_ENABLED:true DD_CLUSTER_AGENT_KUBERNETES_SERVICE_NAME:datadog-cluster-agent DD_CLUSTER_NAME:mini DD_DOGSTATSD_NON_LOCAL_TRAFFIC:true DD_DOGSTATSD_PORT:8125 DD_DOGSTATSD_SOCKET:/var/run/datadog/dsd.socket DD_EXPVAR_PORT:6000 DD_EXTRA_CONFIG_PROVIDERS:clusterchecks endpointschecks DD_HEALTH_PORT:5555 DD_IGNORE_AUTOCONF:etcd kube_controller_manager kube_scheduler kubernetes_state DD_KUBELET_TLS_VERIFY:false DD_KUBERNETES_KUBELET_HOST:192.168.49.2 DD_LOGS_CONFIG_AUTO_MULTI_LINE_DETECTION:false DD_LOGS_CONFIG_CONTAINER_COLLECT_ALL:false DD_LOGS_CONFIG_K8S_CONTAINER_USE_FILE:true DD_LOGS_ENABLED:true DD_LOG_LEVEL:INFO DD_PYTHON_VERSION:3 DD_SECRET_BACKEND_ARGUMENTS:/etc/secret-volume DD_SECRET_BACKEND_COMMAND:/readsecret_multiple_providers.sh DD_SECRET_BACKEND_COMMAND_ALLOW_GROUP_EXEC_PERM:true DEBIAN_FRONTEND:noninteractive DOCKER_DD_AGENT:true GODEBUG:x509ignoreCN=0 KUBERNETES:yes KUBERNETES_PORT:tcp://10.96.0.1:443 KUBERNETES_PORT_443_TCP:tcp://10.96.0.1:443 KUBERNETES_PORT_443_TCP_ADDR:10.96.0.1 KUBERNETES_PORT_443_TCP_PORT:443 KUBERNETES_PORT_443_TCP_PROTO:tcp KUBERNETES_SERVICE_HOST:10.96.0.1 KUBERNETES_SERVICE_PORT:443 KUBERNETES_SERVICE_PORT_HTTPS:443 LD_PRELOAD:/lib/x86_64-linux-gnu/nosys.so PATH:/opt/datadog-agent/bin/agent/:/opt/datadog-agent/embedded/bin/:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin S6_BEHAVIOUR_IF_STAGE2_FAILS:2 S6_KEEP_ENV:1 S6_LOGGING:0 S6_READ_ONLY_ROOT:1] datadog-lzcr8 {docker-pullable:********@sha256:4992b38387ca6dbb732f3493ed66cdf5efe7e08154b7e9e4159b1735dbaabe3f gcr.io/datadoghq/agent:7.41.1 gcr.io/datadoghq/agent agent 7.41.1} map[] 3317261 [{dogstatsdport 8125 UDP} { 8125 udp} { 8126 tcp}] docker {true running unknown 2023-01-27 16:32:21.734951171 +0000 UTC 2023-01-27 16:32:21.838310713 +0000 UTC 0001-01-01 00:00:00 +0000 UTC <nil>} []} failed, metrics may be missing, err: no process found inside this cgroup, impossible to gather network stats
2023-01-27 16:37:55 UTC | CORE | DEBUG | (pkg/collector/worker/check_logger.go:59 in CheckFinished) | check:container | Done running check
2023-01-27 16:37:56 UTC | CORE | DEBUG | (pkg/autodiscovery/config_poller.go:170 in collectOnce) | endpoints-checks provider: no configuration change
2023-01-27 16:37:56 UTC | CORE | DEBUG | (pkg/autodiscovery/config_poller.go:154 in poll) | No modifications in the templates stored in cluster-checks configuration provider
2023-01-27 16:37:57 UTC | CORE | DEBUG | (pkg/collector/worker/check_logger.go:42 in CheckStarted) | check:coredns | Running check...
2023-01-27 16:37:57 UTC | CORE | DEBUG | (pkg/collector/python/check.go:84 in runCheck) | Running python check coredns (version: '2.3.0', id: 'coredns:435d3ef1baaaec95')
2023-01-27 16:37:57 UTC | CORE | DEBUG | (pkg/collector/worker/check_logger.go:59 in CheckFinished) | check:coredns | Done running check
2023-01-27 16:37:58 UTC | CORE | DEBUG | (pkg/collector/worker/check_logger.go:42 in CheckStarted) | check:uptime | Running check...
2023-01-27 16:37:58 UTC | CORE | DEBUG | (pkg/collector/worker/check_logger.go:59 in CheckFinished) | check:uptime | Done running check
2023-01-27 16:37:59 UTC | CORE | DEBUG | (pkg/collector/worker/check_logger.go:42 in CheckStarted) | check:memory | Running check...
2023-01-27 16:37:59 UTC | CORE | DEBUG | (pkg/collector/worker/check_logger.go:59 in CheckFinished) | check:memory | Done running check
2023-01-27 16:37:59 UTC | CORE | DEBUG | (pkg/util/kubernetes/kubelet/podwatcher.go:147 in computeChanges) | Found 0 changed pods out of 10
2023-01-27 16:38:00 UTC | CORE | DEBUG | (pkg/collector/worker/check_logger.go:42 in CheckStarted) | check:io | Running check...
2023-01-27 16:38:00 UTC | CORE | DEBUG | (pkg/collector/worker/check_logger.go:59 in CheckFinished) | check:io | Done running check
2023-01-27 16:38:01 UTC | CORE | DEBUG | (pkg/collector/worker/check_logger.go:42 in CheckStarted) | check:docker | Running check...
2023-01-27 16:38:01 UTC | CORE | DEBUG | (pkg/collector/corechecks/containers/generic/processor.go:116 in Run) | OpenFiles count for: &{{container 59f8054dd8439dda298171fecb121928e077dd0e81835fdfbbd86dd47be7e925} {agent  map[] map[annotation.io.kubernetes.container.hash:b7d4b3a9 annotation.io.kubernetes.container.ports:[{"name":"dogstatsdport","containerPort":8125,"protocol":"UDP"}] annotation.io.kubernetes.container.restartCount:0 annotation.io.kubernetes.container.terminationMessagePath:/dev/termination-log annotation.io.kubernetes.container.terminationMessagePolicy:File annotation.io.kubernetes.pod.terminationGracePeriod:30 io.kubernetes.container.logpath:/var/log/pods/default_datadog-lzcr8_3532449e-51ba-4864-a468-d7a4da5b094f/agent/0.log io.kubernetes.container.name:agent io.kubernetes.docker.type:container io.kubernetes.pod.name:datadog-lzcr8 io.kubernetes.pod.namespace:default io.kubernetes.pod.uid:3532449e-51ba-4864-a468-d7a4da5b094f io.kubernetes.sandbox.id:76d45bcbf4a6c61636d13d5c4c30ac05dcb850055dc7eed90a54e246fe6695bc maintainer:Datadog <package@datadoghq.com>]} map[CURL_CA_BUNDLE:/opt/datadog-agent/embedded/ssl/certs/cacert.pem DATADOG_CLUSTER_AGENT_ADMISSION_CONTROLLER_PORT:tcp://10.98.222.107:443 DATADOG_CLUSTER_AGENT_ADMISSION_CONTROLLER_PORT_443_TCP:tcp://10.98.222.107:443 DATADOG_CLUSTER_AGENT_ADMISSION_CONTROLLER_PORT_443_TCP_ADDR:10.98.222.107 DATADOG_CLUSTER_AGENT_ADMISSION_CONTROLLER_PORT_443_TCP_PORT:443 DATADOG_CLUSTER_AGENT_ADMISSION_CONTROLLER_PORT_443_TCP_PROTO:tcp DATADOG_CLUSTER_AGENT_ADMISSION_CONTROLLER_SERVICE_HOST:10.98.222.107 DATADOG_CLUSTER_AGENT_ADMISSION_CONTROLLER_SERVICE_PORT:443 DATADOG_CLUSTER_AGENT_PORT:tcp://10.104.229.207:5005 DATADOG_CLUSTER_AGENT_PORT_5005_TCP:tcp://10.104.229.207:5005 DATADOG_CLUSTER_AGENT_PORT_5005_TCP_ADDR:10.104.229.207 DATADOG_CLUSTER_AGENT_PORT_5005_TCP_PORT:5005 DATADOG_CLUSTER_AGENT_PORT_5005_TCP_PROTO:tcp DATADOG_CLUSTER_AGENT_SERVICE_HOST:10.104.229.207 DATADOG_CLUSTER_AGENT_SERVICE_PORT:5005 DATADOG_CLUSTER_AGENT_SERVICE_PORT_AGENTPORT:5005 DATADOG_PORT:udp://10.96.246.183:8125 DATADOG_PORT_8125_UDP:udp://10.96.246.183:8125 DATADOG_PORT_8125_UDP_ADDR:10.96.246.183 DATADOG_PORT_8125_UDP_PORT:8125 DATADOG_PORT_8125_UDP_PROTO:udp DATADOG_SERVICE_HOST:10.96.246.183 DATADOG_SERVICE_PORT:8125 DATADOG_SERVICE_PORT_DOGSTATSD:8125 DD_API_KEY:"***************************08bfc" DD_APM_ENABLED:false DD_CLUSTER_AGENT_AUTH_TOKEN:4tcpr3fNUzQmp1frQEqVx8EOLe8LN2fP DD_CLUSTER_AGENT_ENABLED:true DD_CLUSTER_AGENT_KUBERNETES_SERVICE_NAME:datadog-cluster-agent DD_CLUSTER_NAME:mini DD_DOGSTATSD_NON_LOCAL_TRAFFIC:true DD_DOGSTATSD_PORT:8125 DD_DOGSTATSD_SOCKET:/var/run/datadog/dsd.socket DD_EXPVAR_PORT:6000 DD_EXTRA_CONFIG_PROVIDERS:clusterchecks endpointschecks DD_HEALTH_PORT:5555 DD_IGNORE_AUTOCONF:etcd kube_controller_manager kube_scheduler kubernetes_state DD_KUBELET_TLS_VERIFY:false DD_KUBERNETES_KUBELET_HOST:192.168.49.2 DD_LOGS_CONFIG_AUTO_MULTI_LINE_DETECTION:false DD_LOGS_CONFIG_CONTAINER_COLLECT_ALL:false DD_LOGS_CONFIG_K8S_CONTAINER_USE_FILE:true DD_LOGS_ENABLED:true DD_LOG_LEVEL:INFO DD_PYTHON_VERSION:3 DD_SECRET_BACKEND_ARGUMENTS:/etc/secret-volume DD_SECRET_BACKEND_COMMAND:/readsecret_multiple_providers.sh DD_SECRET_BACKEND_COMMAND_ALLOW_GROUP_EXEC_PERM:true DEBIAN_FRONTEND:noninteractive DOCKER_DD_AGENT:true GODEBUG:x509ignoreCN=0 KUBERNETES:yes KUBERNETES_PORT:tcp://10.96.0.1:443 KUBERNETES_PORT_443_TCP:tcp://10.96.0.1:443 KUBERNETES_PORT_443_TCP_ADDR:10.96.0.1 KUBERNETES_PORT_443_TCP_PORT:443 KUBERNETES_PORT_443_TCP_PROTO:tcp KUBERNETES_SERVICE_HOST:10.96.0.1 KUBERNETES_SERVICE_PORT:443 KUBERNETES_SERVICE_PORT_HTTPS:443 LD_PRELOAD:/lib/x86_64-linux-gnu/nosys.so PATH:/opt/datadog-agent/bin/agent/:/opt/datadog-agent/embedded/bin/:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin S6_BEHAVIOUR_IF_STAGE2_FAILS:2 S6_KEEP_ENV:1 S6_LOGGING:0 S6_READ_ONLY_ROOT:1] datadog-lzcr8 {docker-pullable:********@sha256:4992b38387ca6dbb732f3493ed66cdf5efe7e08154b7e9e4159b1735dbaabe3f gcr.io/datadoghq/agent:7.41.1 gcr.io/datadoghq/agent agent 7.41.1} map[] 3317261 [{dogstatsdport 8125 UDP} { 8125 udp} { 8126 tcp}] docker {true running unknown 2023-01-27 16:32:21.734951171 +0000 UTC 2023-01-27 16:32:21.838310713 +0000 UTC 0001-01-01 00:00:00 +0000 UTC <nil>} []} not available through collector "system", err: unable to read any PID open FDs for cgroup id: 59f8054dd8439dda298171fecb121928e077dd0e81835fdfbbd86dd47be7e925. Unable to get count of open files
2023-01-27 16:38:01 UTC | CORE | DEBUG | (pkg/collector/corechecks/containers/docker/check_network.go:95 in Process) | Gathering network metrics for container: &{{container 59f8054dd8439dda298171fecb121928e077dd0e81835fdfbbd86dd47be7e925} {agent  map[] map[annotation.io.kubernetes.container.hash:b7d4b3a9 annotation.io.kubernetes.container.ports:[{"name":"dogstatsdport","containerPort":8125,"protocol":"UDP"}] annotation.io.kubernetes.container.restartCount:0 annotation.io.kubernetes.container.terminationMessagePath:/dev/termination-log annotation.io.kubernetes.container.terminationMessagePolicy:File annotation.io.kubernetes.pod.terminationGracePeriod:30 io.kubernetes.container.logpath:/var/log/pods/default_datadog-lzcr8_3532449e-51ba-4864-a468-d7a4da5b094f/agent/0.log io.kubernetes.container.name:agent io.kubernetes.docker.type:container io.kubernetes.pod.name:datadog-lzcr8 io.kubernetes.pod.namespace:default io.kubernetes.pod.uid:3532449e-51ba-4864-a468-d7a4da5b094f io.kubernetes.sandbox.id:76d45bcbf4a6c61636d13d5c4c30ac05dcb850055dc7eed90a54e246fe6695bc maintainer:Datadog <package@datadoghq.com>]} map[CURL_CA_BUNDLE:/opt/datadog-agent/embedded/ssl/certs/cacert.pem DATADOG_CLUSTER_AGENT_ADMISSION_CONTROLLER_PORT:tcp://10.98.222.107:443 DATADOG_CLUSTER_AGENT_ADMISSION_CONTROLLER_PORT_443_TCP:tcp://10.98.222.107:443 DATADOG_CLUSTER_AGENT_ADMISSION_CONTROLLER_PORT_443_TCP_ADDR:10.98.222.107 DATADOG_CLUSTER_AGENT_ADMISSION_CONTROLLER_PORT_443_TCP_PORT:443 DATADOG_CLUSTER_AGENT_ADMISSION_CONTROLLER_PORT_443_TCP_PROTO:tcp DATADOG_CLUSTER_AGENT_ADMISSION_CONTROLLER_SERVICE_HOST:10.98.222.107 DATADOG_CLUSTER_AGENT_ADMISSION_CONTROLLER_SERVICE_PORT:443 DATADOG_CLUSTER_AGENT_PORT:tcp://10.104.229.207:5005 DATADOG_CLUSTER_AGENT_PORT_5005_TCP:tcp://10.104.229.207:5005 DATADOG_CLUSTER_AGENT_PORT_5005_TCP_ADDR:10.104.229.207 DATADOG_CLUSTER_AGENT_PORT_5005_TCP_PORT:5005 DATADOG_CLUSTER_AGENT_PORT_5005_TCP_PROTO:tcp DATADOG_CLUSTER_AGENT_SERVICE_HOST:10.104.229.207 DATADOG_CLUSTER_AGENT_SERVICE_PORT:5005 DATADOG_CLUSTER_AGENT_SERVICE_PORT_AGENTPORT:5005 DATADOG_PORT:udp://10.96.246.183:8125 DATADOG_PORT_8125_UDP:udp://10.96.246.183:8125 DATADOG_PORT_8125_UDP_ADDR:10.96.246.183 DATADOG_PORT_8125_UDP_PORT:8125 DATADOG_PORT_8125_UDP_PROTO:udp DATADOG_SERVICE_HOST:10.96.246.183 DATADOG_SERVICE_PORT:8125 DATADOG_SERVICE_PORT_DOGSTATSD:8125 DD_API_KEY:"***************************08bfc" DD_APM_ENABLED:false DD_CLUSTER_AGENT_AUTH_TOKEN:4tcpr3fNUzQmp1frQEqVx8EOLe8LN2fP DD_CLUSTER_AGENT_ENABLED:true DD_CLUSTER_AGENT_KUBERNETES_SERVICE_NAME:datadog-cluster-agent DD_CLUSTER_NAME:mini DD_DOGSTATSD_NON_LOCAL_TRAFFIC:true DD_DOGSTATSD_PORT:8125 DD_DOGSTATSD_SOCKET:/var/run/datadog/dsd.socket DD_EXPVAR_PORT:6000 DD_EXTRA_CONFIG_PROVIDERS:clusterchecks endpointschecks DD_HEALTH_PORT:5555 DD_IGNORE_AUTOCONF:etcd kube_controller_manager kube_scheduler kubernetes_state DD_KUBELET_TLS_VERIFY:false DD_KUBERNETES_KUBELET_HOST:192.168.49.2 DD_LOGS_CONFIG_AUTO_MULTI_LINE_DETECTION:false DD_LOGS_CONFIG_CONTAINER_COLLECT_ALL:false DD_LOGS_CONFIG_K8S_CONTAINER_USE_FILE:true DD_LOGS_ENABLED:true DD_LOG_LEVEL:INFO DD_PYTHON_VERSION:3 DD_SECRET_BACKEND_ARGUMENTS:/etc/secret-volume DD_SECRET_BACKEND_COMMAND:/readsecret_multiple_providers.sh DD_SECRET_BACKEND_COMMAND_ALLOW_GROUP_EXEC_PERM:true DEBIAN_FRONTEND:noninteractive DOCKER_DD_AGENT:true GODEBUG:x509ignoreCN=0 KUBERNETES:yes KUBERNETES_PORT:tcp://10.96.0.1:443 KUBERNETES_PORT_443_TCP:tcp://10.96.0.1:443 KUBERNETES_PORT_443_TCP_ADDR:10.96.0.1 KUBERNETES_PORT_443_TCP_PORT:443 KUBERNETES_PORT_443_TCP_PROTO:tcp KUBERNETES_SERVICE_HOST:10.96.0.1 KUBERNETES_SERVICE_PORT:443 KUBERNETES_SERVICE_PORT_HTTPS:443 LD_PRELOAD:/lib/x86_64-linux-gnu/nosys.so PATH:/opt/datadog-agent/bin/agent/:/opt/datadog-agent/embedded/bin/:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin S6_BEHAVIOUR_IF_STAGE2_FAILS:2 S6_KEEP_ENV:1 S6_LOGGING:0 S6_READ_ONLY_ROOT:1] datadog-lzcr8 {docker-pullable:********@sha256:4992b38387ca6dbb732f3493ed66cdf5efe7e08154b7e9e4159b1735dbaabe3f gcr.io/datadoghq/agent:7.41.1 gcr.io/datadoghq/agent agent 7.41.1} map[] 3317261 [{dogstatsdport 8125 UDP} { 8125 udp} { 8126 tcp}] docker {true running unknown 2023-01-27 16:32:21.734951171 +0000 UTC 2023-01-27 16:32:21.838310713 +0000 UTC 0001-01-01 00:00:00 +0000 UTC <nil>} []} failed, metrics may be missing, err: no process found inside this cgroup, impossible to gather network stats
2023-01-27 16:38:01 UTC | CORE | DEBUG | (pkg/collector/corechecks/containers/generic/processor.go:116 in Run) | OpenFiles count for: &{{container 0845c059f2b70949ddc96aa1d7160b0e539fc73cd5e1614c187dfb509a205a8e} {redis  map[] map[io.kubernetes.pod.namespace:default]} map[]  {docker-pullable:********@sha256:325d5a448d8f6c1d30a0a0fb26090343279d4cf23258b26b1745862f332e9479 redis:latest redis redis latest} map[] 0 [{ 6379 TCP}] docker {true running  2023-01-27 16:34:28 +0000 UTC 2023-01-27 16:34:28 +0000 UTC 0001-01-01 00:00:00 +0000 UTC <nil>} []} not available through collector "system", err: unable to read any PID open FDs for cgroup id: 0845c059f2b70949ddc96aa1d7160b0e539fc73cd5e1614c187dfb509a205a8e. Unable to get count of open files
2023-01-27 16:38:01 UTC | CORE | DEBUG | (pkg/collector/corechecks/containers/docker/check_network.go:95 in Process) | Gathering network metrics for container: &{{container 0845c059f2b70949ddc96aa1d7160b0e539fc73cd5e1614c187dfb509a205a8e} {redis  map[] map[io.kubernetes.pod.namespace:default]} map[]  {docker-pullable:********@sha256:325d5a448d8f6c1d30a0a0fb26090343279d4cf23258b26b1745862f332e9479 redis:latest redis redis latest} map[] 0 [{ 6379 TCP}] docker {true running  2023-01-27 16:34:28 +0000 UTC 2023-01-27 16:34:28 +0000 UTC 0001-01-01 00:00:00 +0000 UTC <nil>} []} failed, metrics may be missing, err: no process found inside this cgroup, impossible to gather network stats
2023-01-27 16:38:01 UTC | CORE | DEBUG | (pkg/collector/corechecks/containers/docker/check_network.go:226 in findDockerNetworks) | No PID found for container: e20ff6db4165dfd5aabff2ed6f58b8f4fc1eae1174ca56e1204efe6ea7586bc1, skipping network
2023-01-27 16:38:01 UTC | CORE | DEBUG | (pkg/collector/corechecks/containers/docker/check_network.go:226 in findDockerNetworks) | No PID found for container: 76d45bcbf4a6c61636d13d5c4c30ac05dcb850055dc7eed90a54e246fe6695bc, skipping network
2023-01-27 16:38:01 UTC | CORE | DEBUG | (pkg/collector/corechecks/containers/docker/check_network.go:226 in findDockerNetworks) | No PID found for container: 8dd818c41c9f074b211c5c2bb6fc28d95247802845af0e8ff083a894d314313a, skipping network
2023-01-27 16:38:01 UTC | CORE | DEBUG | (pkg/collector/corechecks/containers/docker/check_network.go:205 in findDockerNetworks) | Container e8f8042ab20dab731de6c74db22687addb7e9529f8bd132e36f4596d0ebf418a is in network host mode, its network metrics are for the whole host
2023-01-27 16:38:01 UTC | CORE | DEBUG | (pkg/collector/corechecks/containers/docker/check_network.go:205 in findDockerNetworks) | Container f55f65aaa1b4bba2f21a9147e18b01bc0cb44ca7c789e2327525145e87b34be7 is in network host mode, its network metrics are for the whole host
2023-01-27 16:38:01 UTC | CORE | DEBUG | (pkg/collector/corechecks/containers/docker/check_network.go:226 in findDockerNetworks) | No PID found for container: a7eddfe2282b3779c113866c707e8c5ddd904f9e97ba107daeab92699d5c8d48, skipping network
2023-01-27 16:38:01 UTC | CORE | DEBUG | (pkg/collector/corechecks/containers/docker/check_network.go:226 in findDockerNetworks) | No PID found for container: 3a8b4db967a630c33d1e9556f11868a15de9b7141c24c03f70a9b05289229cbb, skipping network
2023-01-27 16:38:01 UTC | CORE | DEBUG | (pkg/collector/corechecks/containers/docker/check_network.go:205 in findDockerNetworks) | Container eaa983a0a2063c2c4135e206f8a78ea1ae5cbe130099e292733a8973cbfcd35a is in network host mode, its network metrics are for the whole host
2023-01-27 16:38:01 UTC | CORE | DEBUG | (pkg/collector/corechecks/containers/docker/check_network.go:205 in findDockerNetworks) | Container 5377f5f00a9e410217525941c79154ad0010a94179fd3d7fcc7fcb3ca9b331dd is in network host mode, its network metrics are for the whole host
2023-01-27 16:38:01 UTC | CORE | DEBUG | (pkg/collector/corechecks/containers/docker/check_network.go:205 in findDockerNetworks) | Container 94dfb30dd3bc339ce158d621fb9fd4c3b50feb7e4725609850a717fa2bf25fbb is in network host mode, its network metrics are for the whole host
2023-01-27 16:38:01 UTC | CORE | DEBUG | (pkg/collector/corechecks/containers/docker/check_network.go:205 in findDockerNetworks) | Container f7b4614499df44006c7a7b2fddd15d71e3b1af72c7447601e6bdb74a8485be3c is in network host mode, its network metrics are for the whole host
2023-01-27 16:38:01 UTC | CORE | DEBUG | (pkg/collector/worker/check_logger.go:59 in CheckFinished) | check:docker | Done running check
2023-01-27 16:38:02 UTC | CORE | DEBUG | (pkg/collector/worker/check_logger.go:42 in CheckStarted) | check:cpu | Running check...
2023-01-27 16:38:02 UTC | CORE | DEBUG | (pkg/collector/worker/check_logger.go:59 in CheckFinished) | check:cpu | Done running check
2023-01-27 16:38:03 UTC | CORE | DEBUG | (pkg/collector/worker/check_logger.go:42 in CheckStarted) | check:kube_apiserver_metrics | Running check...
2023-01-27 16:38:03 UTC | CORE | DEBUG | (pkg/collector/python/check.go:84 in runCheck) | Running python check kube_apiserver_metrics (version: '3.3.0', id: 'kube_apiserver_metrics:9870dafe14e0de84')
2023-01-27 16:38:03 UTC | CORE | DEBUG | (pkg/collector/worker/check_logger.go:59 in CheckFinished) | check:kube_apiserver_metrics | Done running check
2023-01-27 16:38:04 UTC | CORE | DEBUG | (pkg/collector/worker/check_logger.go:42 in CheckStarted) | check:datadog_cluster_agent | Running check...
2023-01-27 16:38:04 UTC | CORE | DEBUG | (pkg/collector/python/check.go:84 in runCheck) | Running python check datadog_cluster_agent (version: '2.4.0', id: 'datadog_cluster_agent:37b5fa98a5978a46')
2023-01-27 16:38:04 UTC | CORE | DEBUG | (pkg/collector/worker/check_logger.go:59 in CheckFinished) | check:datadog_cluster_agent | Done running check
2023-01-27 16:38:04 UTC | CORE | DEBUG | (pkg/util/kubernetes/kubelet/podwatcher.go:147 in computeChanges) | Found 0 changed pods out of 10
2023-01-27 16:38:05 UTC | CORE | DEBUG | (pkg/logs/internal/launchers/file/launcher.go:141 in scan) | Scan - got 0 files from FilesToTail and currently tailing 0 files
2023-01-27 16:38:05 UTC | CORE | DEBUG | (pkg/logs/internal/launchers/file/launcher.go:195 in scan) | After stopping tailers, there are 0 tailers running.
2023-01-27 16:38:05 UTC | CORE | DEBUG | (pkg/logs/internal/launchers/file/launcher.go:212 in scan) | After starting new tailers, there are 0 tailers running. Limit is 500.
2023-01-27 16:38:06 UTC | CORE | DEBUG | (pkg/collector/worker/check_logger.go:42 in CheckStarted) | check:network | Running check...
2023-01-27 16:38:06 UTC | CORE | DEBUG | (pkg/collector/python/check.go:84 in runCheck) | Running python check network (version: '2.9.2', id: 'network:d884b5186b651429')
2023-01-27 16:38:06 UTC | CORE | DEBUG | (pkg/collector/worker/check_logger.go:59 in CheckFinished) | check:network | Done running check
2023-01-27 16:38:06 UTC | CORE | DEBUG | (pkg/collector/worker/check_logger.go:42 in CheckStarted) | check:kubelet | Running check...
2023-01-27 16:38:06 UTC | CORE | DEBUG | (pkg/collector/python/check.go:84 in runCheck) | Running python check kubelet (version: '7.5.1', id: 'kubelet:5bbc63f3938c02f4')
2023-01-27 16:38:06 UTC | CORE | DEBUG | (pkg/collector/python/kubeutil.go:62 in GetKubeletConnectionInfo) | cache hit for kubelet connection info
2023-01-27 16:38:06 UTC | CORE | DEBUG | (pkg/autodiscovery/config_poller.go:170 in collectOnce) | endpoints-checks provider: no configuration change
2023-01-27 16:38:06 UTC | CORE | DEBUG | (pkg/autodiscovery/config_poller.go:154 in poll) | No modifications in the templates stored in cluster-checks configuration provider
2023-01-27 16:38:07 UTC | CORE | DEBUG | (pkg/collector/worker/check_logger.go:42 in CheckStarted) | check:load | Running check...
2023-01-27 16:38:07 UTC | CORE | DEBUG | (pkg/collector/worker/check_logger.go:59 in CheckFinished) | check:load | Done running check
2023-01-27 16:38:08 UTC | CORE | DEBUG | (pkg/collector/worker/check_logger.go:42 in CheckStarted) | check:file_handle | Running check...
2023-01-27 16:38:08 UTC | CORE | DEBUG | (pkg/collector/worker/check_logger.go:59 in CheckFinished) | check:file_handle | Done running check
2023-01-27 16:38:09 UTC | CORE | DEBUG | (pkg/collector/worker/check_logger.go:42 in CheckStarted) | check:disk | Running check...
2023-01-27 16:38:09 UTC | CORE | DEBUG | (pkg/collector/python/check.go:84 in runCheck) | Running python check disk (version: '4.7.1', id: 'disk:e5dffb8bef24336f')
2023-01-27 16:38:09 UTC | CORE | DEBUG | (pkg/collector/worker/check_logger.go:59 in CheckFinished) | check:disk | Done running check
2023-01-27 16:38:09 UTC | CORE | DEBUG | (pkg/util/kubernetes/kubelet/podwatcher.go:147 in computeChanges) | Found 1 changed pods out of 11
2023-01-27 16:38:09 UTC | CORE | DEBUG | (pkg/autodiscovery/providers/kubecontainer.go:192 in generateConfig) | Pod "redis" has reference to non-existing container "0845c059f2b70949ddc96aa1d7160b0e539fc73cd5e1614c187dfb509a205a8e"
2023-01-27 16:38:09 UTC | CORE | DEBUG | (pkg/autodiscovery/listeners/service.go:171 in filterTemplatesOverriddenChecks) | Ignoring config from file:/etc/datadog-agent/conf.d/redisdb.d/auto_conf.yaml: the service docker://87626279d1c48b74dc70964bafa8935d0555a11ec798c6d028921ae8841b1553 overrides check redisdb
2023-01-27 16:38:09 UTC | CORE | INFO | (pkg/autodiscovery/config_poller.go:105 in stream) | kubernetes-container-allinone provider: collected 1 new configurations, removed 1
2023-01-27 16:38:09 UTC | CORE | DEBUG | (pkg/tagger/collectors/workloadmeta_extract.go:304 in handleKubePod) | cannot extract tags from pod container: pod "redis" has reference to non-existing container "0845c059f2b70949ddc96aa1d7160b0e539fc73cd5e1614c187dfb509a205a8e"
2023-01-27 16:38:09 UTC | CORE | DEBUG | (pkg/autodiscovery/listeners/service.go:171 in filterTemplatesOverriddenChecks) | Ignoring config from file:/etc/datadog-agent/conf.d/redisdb.d/auto_conf.yaml: the service docker://87626279d1c48b74dc70964bafa8935d0555a11ec798c6d028921ae8841b1553 overrides check redisdb
2023-01-27 16:38:09 UTC | CORE | DEBUG | (pkg/autodiscovery/configresolver/configresolver.go:519 in getHost) | Network "" not found, trying bridge IP instead
2023-01-27 16:38:09 UTC | CORE | DEBUG | (pkg/secrets/fetch_secret.go:74 in execCommand) | 2023-01-27 16:38:09.257811763 +0000 UTC m=+347.354020327 | calling secret_backend_command with payload: '{"secrets":["redis_password"],"version":"1.0"}'
2023-01-27 16:38:09 UTC | CORE | DEBUG | (pkg/secrets/fetch_secret.go:78 in execCommand) | 2023-01-27 16:38:09.35982443 +0000 UTC m=+347.456032993 | secret_backend_command '/readsecret_multiple_providers.sh' completed in 101.931541ms
2023-01-27 16:38:09 UTC | CORE | DEBUG | (pkg/secrets/fetch_secret.go:100 in execCommand) | secret_backend_command stderr: 
2023-01-27 16:38:09 UTC | CORE | DEBUG | (pkg/autodiscovery/autoconfig.go:143 in checkTagFreshness) | Tags changed for service kubernetes_pod_uid://2bb19de7-ac14-4216-8814-70afb0d89ad6, rescheduling associated checks if any
2023-01-27 16:38:09 UTC | CORE | DEBUG | (pkg/autodiscovery/autoconfig.go:143 in checkTagFreshness) | Tags changed for service container_id://87626279d1c48b74dc70964bafa8935d0555a11ec798c6d028921ae8841b1553, rescheduling associated checks if any
2023-01-27 16:38:09 UTC | CORE | DEBUG | (pkg/autodiscovery/listeners/service.go:171 in filterTemplatesOverriddenChecks) | Ignoring config from file:/etc/datadog-agent/conf.d/redisdb.d/auto_conf.yaml: the service docker://87626279d1c48b74dc70964bafa8935d0555a11ec798c6d028921ae8841b1553 overrides check redisdb
2023-01-27 16:38:09 UTC | CORE | DEBUG | (pkg/autodiscovery/configresolver/configresolver.go:519 in getHost) | Network "" not found, trying bridge IP instead
2023-01-27 16:38:09 UTC | CORE | DEBUG | (pkg/secrets/fetch_secret.go:74 in execCommand) | 2023-01-27 16:38:09.987868638 +0000 UTC m=+348.084077202 | calling secret_backend_command with payload: '{"secrets":["redis_password"],"version":"1.0"}'
2023-01-27 16:38:10 UTC | CORE | DEBUG | (pkg/aggregator/aggregator.go:659 in sendEvents) | Flushing 1 events to the forwarder
2023-01-27 16:38:10 UTC | CORE | DEBUG | (pkg/aggregator/demultiplexer.go:144 in sendIterableSeries) | Demultiplexer: sendIterableSeries: start sending iterable series to the serializer
2023-01-27 16:38:10 UTC | CORE | DEBUG | (pkg/aggregator/aggregator.go:604 in sendServiceChecks) | Flushing 22 service checks to the forwarder
2023-01-27 16:38:10 UTC | CORE | DEBUG | (pkg/secrets/fetch_secret.go:78 in execCommand) | 2023-01-27 16:38:10.081081472 +0000 UTC m=+348.177289994 | secret_backend_command '/readsecret_multiple_providers.sh' completed in 93.13325ms
2023-01-27 16:38:10 UTC | CORE | DEBUG | (pkg/secrets/fetch_secret.go:100 in execCommand) | secret_backend_command stderr: 
2023-01-27 16:38:10 UTC | CORE | DEBUG | (pkg/aggregator/demultiplexer.go:150 in sendIterableSeries) | Demultiplexer: sendIterableSeries: stop routine
2023-01-27 16:38:10 UTC | CORE | DEBUG | (pkg/collector/worker/check_logger.go:42 in CheckStarted) | check:container | Running check...
2023-01-27 16:38:10 UTC | CORE | DEBUG | (pkg/collector/corechecks/containers/generic/processor.go:116 in Run) | OpenFiles count for: &{{container 59f8054dd8439dda298171fecb121928e077dd0e81835fdfbbd86dd47be7e925} {agent  map[] map[annotation.io.kubernetes.container.hash:b7d4b3a9 annotation.io.kubernetes.container.ports:[{"name":"dogstatsdport","containerPort":8125,"protocol":"UDP"}] annotation.io.kubernetes.container.restartCount:0 annotation.io.kubernetes.container.terminationMessagePath:/dev/termination-log annotation.io.kubernetes.container.terminationMessagePolicy:File annotation.io.kubernetes.pod.terminationGracePeriod:30 io.kubernetes.container.logpath:/var/log/pods/default_datadog-lzcr8_3532449e-51ba-4864-a468-d7a4da5b094f/agent/0.log io.kubernetes.container.name:agent io.kubernetes.docker.type:container io.kubernetes.pod.name:datadog-lzcr8 io.kubernetes.pod.namespace:default io.kubernetes.pod.uid:3532449e-51ba-4864-a468-d7a4da5b094f io.kubernetes.sandbox.id:76d45bcbf4a6c61636d13d5c4c30ac05dcb850055dc7eed90a54e246fe6695bc maintainer:Datadog <package@datadoghq.com>]} map[CURL_CA_BUNDLE:/opt/datadog-agent/embedded/ssl/certs/cacert.pem DATADOG_CLUSTER_AGENT_ADMISSION_CONTROLLER_PORT:tcp://10.98.222.107:443 DATADOG_CLUSTER_AGENT_ADMISSION_CONTROLLER_PORT_443_TCP:tcp://10.98.222.107:443 DATADOG_CLUSTER_AGENT_ADMISSION_CONTROLLER_PORT_443_TCP_ADDR:10.98.222.107 DATADOG_CLUSTER_AGENT_ADMISSION_CONTROLLER_PORT_443_TCP_PORT:443 DATADOG_CLUSTER_AGENT_ADMISSION_CONTROLLER_PORT_443_TCP_PROTO:tcp DATADOG_CLUSTER_AGENT_ADMISSION_CONTROLLER_SERVICE_HOST:10.98.222.107 DATADOG_CLUSTER_AGENT_ADMISSION_CONTROLLER_SERVICE_PORT:443 DATADOG_CLUSTER_AGENT_PORT:tcp://10.104.229.207:5005 DATADOG_CLUSTER_AGENT_PORT_5005_TCP:tcp://10.104.229.207:5005 DATADOG_CLUSTER_AGENT_PORT_5005_TCP_ADDR:10.104.229.207 DATADOG_CLUSTER_AGENT_PORT_5005_TCP_PORT:5005 DATADOG_CLUSTER_AGENT_PORT_5005_TCP_PROTO:tcp DATADOG_CLUSTER_AGENT_SERVICE_HOST:10.104.229.207 DATADOG_CLUSTER_AGENT_SERVICE_PORT:5005 DATADOG_CLUSTER_AGENT_SERVICE_PORT_AGENTPORT:5005 DATADOG_PORT:udp://10.96.246.183:8125 DATADOG_PORT_8125_UDP:udp://10.96.246.183:8125 DATADOG_PORT_8125_UDP_ADDR:10.96.246.183 DATADOG_PORT_8125_UDP_PORT:8125 DATADOG_PORT_8125_UDP_PROTO:udp DATADOG_SERVICE_HOST:10.96.246.183 DATADOG_SERVICE_PORT:8125 DATADOG_SERVICE_PORT_DOGSTATSD:8125 DD_API_KEY:"***************************08bfc" DD_APM_ENABLED:false DD_CLUSTER_AGENT_AUTH_TOKEN:4tcpr3fNUzQmp1frQEqVx8EOLe8LN2fP DD_CLUSTER_AGENT_ENABLED:true DD_CLUSTER_AGENT_KUBERNETES_SERVICE_NAME:datadog-cluster-agent DD_CLUSTER_NAME:mini DD_DOGSTATSD_NON_LOCAL_TRAFFIC:true DD_DOGSTATSD_PORT:8125 DD_DOGSTATSD_SOCKET:/var/run/datadog/dsd.socket DD_EXPVAR_PORT:6000 DD_EXTRA_CONFIG_PROVIDERS:clusterchecks endpointschecks DD_HEALTH_PORT:5555 DD_IGNORE_AUTOCONF:etcd kube_controller_manager kube_scheduler kubernetes_state DD_KUBELET_TLS_VERIFY:false DD_KUBERNETES_KUBELET_HOST:192.168.49.2 DD_LOGS_CONFIG_AUTO_MULTI_LINE_DETECTION:false DD_LOGS_CONFIG_CONTAINER_COLLECT_ALL:false DD_LOGS_CONFIG_K8S_CONTAINER_USE_FILE:true DD_LOGS_ENABLED:true DD_LOG_LEVEL:INFO DD_PYTHON_VERSION:3 DD_SECRET_BACKEND_ARGUMENTS:/etc/secret-volume DD_SECRET_BACKEND_COMMAND:/readsecret_multiple_providers.sh DD_SECRET_BACKEND_COMMAND_ALLOW_GROUP_EXEC_PERM:true DEBIAN_FRONTEND:noninteractive DOCKER_DD_AGENT:true GODEBUG:x509ignoreCN=0 KUBERNETES:yes KUBERNETES_PORT:tcp://10.96.0.1:443 KUBERNETES_PORT_443_TCP:tcp://10.96.0.1:443 KUBERNETES_PORT_443_TCP_ADDR:10.96.0.1 KUBERNETES_PORT_443_TCP_PORT:443 KUBERNETES_PORT_443_TCP_PROTO:tcp KUBERNETES_SERVICE_HOST:10.96.0.1 KUBERNETES_SERVICE_PORT:443 KUBERNETES_SERVICE_PORT_HTTPS:443 LD_PRELOAD:/lib/x86_64-linux-gnu/nosys.so PATH:/opt/datadog-agent/bin/agent/:/opt/datadog-agent/embedded/bin/:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin S6_BEHAVIOUR_IF_STAGE2_FAILS:2 S6_KEEP_ENV:1 S6_LOGGING:0 S6_READ_ONLY_ROOT:1] datadog-lzcr8 {docker-pullable:********@sha256:4992b38387ca6dbb732f3493ed66cdf5efe7e08154b7e9e4159b1735dbaabe3f gcr.io/datadoghq/agent:7.41.1 gcr.io/datadoghq/agent agent 7.41.1} map[] 3317261 [{dogstatsdport 8125 UDP} { 8125 udp} { 8126 tcp}] docker {true running unknown 2023-01-27 16:32:21.734951171 +0000 UTC 2023-01-27 16:32:21.838310713 +0000 UTC 0001-01-01 00:00:00 +0000 UTC <nil>} []} not available through collector "system", err: unable to read any PID open FDs for cgroup id: 59f8054dd8439dda298171fecb121928e077dd0e81835fdfbbd86dd47be7e925. Unable to get count of open files
2023-01-27 16:38:10 UTC | CORE | DEBUG | (pkg/collector/corechecks/containers/generic/processor_network.go:50 in Process) | Gathering network metrics for container: &{{container 59f8054dd8439dda298171fecb121928e077dd0e81835fdfbbd86dd47be7e925} {agent  map[] map[annotation.io.kubernetes.container.hash:b7d4b3a9 annotation.io.kubernetes.container.ports:[{"name":"dogstatsdport","containerPort":8125,"protocol":"UDP"}] annotation.io.kubernetes.container.restartCount:0 annotation.io.kubernetes.container.terminationMessagePath:/dev/termination-log annotation.io.kubernetes.container.terminationMessagePolicy:File annotation.io.kubernetes.pod.terminationGracePeriod:30 io.kubernetes.container.logpath:/var/log/pods/default_datadog-lzcr8_3532449e-51ba-4864-a468-d7a4da5b094f/agent/0.log io.kubernetes.container.name:agent io.kubernetes.docker.type:container io.kubernetes.pod.name:datadog-lzcr8 io.kubernetes.pod.namespace:default io.kubernetes.pod.uid:3532449e-51ba-4864-a468-d7a4da5b094f io.kubernetes.sandbox.id:76d45bcbf4a6c61636d13d5c4c30ac05dcb850055dc7eed90a54e246fe6695bc maintainer:Datadog <package@datadoghq.com>]} map[CURL_CA_BUNDLE:/opt/datadog-agent/embedded/ssl/certs/cacert.pem DATADOG_CLUSTER_AGENT_ADMISSION_CONTROLLER_PORT:tcp://10.98.222.107:443 DATADOG_CLUSTER_AGENT_ADMISSION_CONTROLLER_PORT_443_TCP:tcp://10.98.222.107:443 DATADOG_CLUSTER_AGENT_ADMISSION_CONTROLLER_PORT_443_TCP_ADDR:10.98.222.107 DATADOG_CLUSTER_AGENT_ADMISSION_CONTROLLER_PORT_443_TCP_PORT:443 DATADOG_CLUSTER_AGENT_ADMISSION_CONTROLLER_PORT_443_TCP_PROTO:tcp DATADOG_CLUSTER_AGENT_ADMISSION_CONTROLLER_SERVICE_HOST:10.98.222.107 DATADOG_CLUSTER_AGENT_ADMISSION_CONTROLLER_SERVICE_PORT:443 DATADOG_CLUSTER_AGENT_PORT:tcp://10.104.229.207:5005 DATADOG_CLUSTER_AGENT_PORT_5005_TCP:tcp://10.104.229.207:5005 DATADOG_CLUSTER_AGENT_PORT_5005_TCP_ADDR:10.104.229.207 DATADOG_CLUSTER_AGENT_PORT_5005_TCP_PORT:5005 DATADOG_CLUSTER_AGENT_PORT_5005_TCP_PROTO:tcp DATADOG_CLUSTER_AGENT_SERVICE_HOST:10.104.229.207 DATADOG_CLUSTER_AGENT_SERVICE_PORT:5005 DATADOG_CLUSTER_AGENT_SERVICE_PORT_AGENTPORT:5005 DATADOG_PORT:udp://10.96.246.183:8125 DATADOG_PORT_8125_UDP:udp://10.96.246.183:8125 DATADOG_PORT_8125_UDP_ADDR:10.96.246.183 DATADOG_PORT_8125_UDP_PORT:8125 DATADOG_PORT_8125_UDP_PROTO:udp DATADOG_SERVICE_HOST:10.96.246.183 DATADOG_SERVICE_PORT:8125 DATADOG_SERVICE_PORT_DOGSTATSD:8125 DD_API_KEY:"***************************08bfc" DD_APM_ENABLED:false DD_CLUSTER_AGENT_AUTH_TOKEN:4tcpr3fNUzQmp1frQEqVx8EOLe8LN2fP DD_CLUSTER_AGENT_ENABLED:true DD_CLUSTER_AGENT_KUBERNETES_SERVICE_NAME:datadog-cluster-agent DD_CLUSTER_NAME:mini DD_DOGSTATSD_NON_LOCAL_TRAFFIC:true DD_DOGSTATSD_PORT:8125 DD_DOGSTATSD_SOCKET:/var/run/datadog/dsd.socket DD_EXPVAR_PORT:6000 DD_EXTRA_CONFIG_PROVIDERS:clusterchecks endpointschecks DD_HEALTH_PORT:5555 DD_IGNORE_AUTOCONF:etcd kube_controller_manager kube_scheduler kubernetes_state DD_KUBELET_TLS_VERIFY:false DD_KUBERNETES_KUBELET_HOST:192.168.49.2 DD_LOGS_CONFIG_AUTO_MULTI_LINE_DETECTION:false DD_LOGS_CONFIG_CONTAINER_COLLECT_ALL:false DD_LOGS_CONFIG_K8S_CONTAINER_USE_FILE:true DD_LOGS_ENABLED:true DD_LOG_LEVEL:INFO DD_PYTHON_VERSION:3 DD_SECRET_BACKEND_ARGUMENTS:/etc/secret-volume DD_SECRET_BACKEND_COMMAND:/readsecret_multiple_providers.sh DD_SECRET_BACKEND_COMMAND_ALLOW_GROUP_EXEC_PERM:true DEBIAN_FRONTEND:noninteractive DOCKER_DD_AGENT:true GODEBUG:x509ignoreCN=0 KUBERNETES:yes KUBERNETES_PORT:tcp://10.96.0.1:443 KUBERNETES_PORT_443_TCP:tcp://10.96.0.1:443 KUBERNETES_PORT_443_TCP_ADDR:10.96.0.1 KUBERNETES_PORT_443_TCP_PORT:443 KUBERNETES_PORT_443_TCP_PROTO:tcp KUBERNETES_SERVICE_HOST:10.96.0.1 KUBERNETES_SERVICE_PORT:443 KUBERNETES_SERVICE_PORT_HTTPS:443 LD_PRELOAD:/lib/x86_64-linux-gnu/nosys.so PATH:/opt/datadog-agent/bin/agent/:/opt/datadog-agent/embedded/bin/:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin S6_BEHAVIOUR_IF_STAGE2_FAILS:2 S6_KEEP_ENV:1 S6_LOGGING:0 S6_READ_ONLY_ROOT:1] datadog-lzcr8 {docker-pullable:********@sha256:4992b38387ca6dbb732f3493ed66cdf5efe7e08154b7e9e4159b1735dbaabe3f gcr.io/datadoghq/agent:7.41.1 gcr.io/datadoghq/agent agent 7.41.1} map[] 3317261 [{dogstatsdport 8125 UDP} { 8125 udp} { 8126 tcp}] docker {true running unknown 2023-01-27 16:32:21.734951171 +0000 UTC 2023-01-27 16:32:21.838310713 +0000 UTC 0001-01-01 00:00:00 +0000 UTC <nil>} []} failed, metrics may be missing, err: no process found inside this cgroup, impossible to gather network stats
2023-01-27 16:38:10 UTC | CORE | DEBUG | (pkg/collector/worker/check_logger.go:59 in CheckFinished) | check:container | Done running check
2023-01-27 16:38:10 UTC | CORE | DEBUG | (pkg/collector/worker/check_logger.go:59 in CheckFinished) | check:kubelet | Done running check
2023-01-27 16:38:12 UTC | CORE | DEBUG | (pkg/collector/worker/check_logger.go:42 in CheckStarted) | check:coredns | Running check...
2023-01-27 16:38:12 UTC | CORE | DEBUG | (pkg/collector/python/check.go:84 in runCheck) | Running python check coredns (version: '2.3.0', id: 'coredns:435d3ef1baaaec95')
2023-01-27 16:38:12 UTC | CORE | DEBUG | (pkg/collector/worker/check_logger.go:59 in CheckFinished) | check:coredns | Done running check
2023-01-27 16:38:13 UTC | CORE | DEBUG | (pkg/collector/worker/check_logger.go:42 in CheckStarted) | check:uptime | Running check...
2023-01-27 16:38:13 UTC | CORE | DEBUG | (pkg/collector/worker/check_logger.go:59 in CheckFinished) | check:uptime | Done running check
2023-01-27 16:38:14 UTC | CORE | DEBUG | (pkg/collector/worker/check_logger.go:42 in CheckStarted) | check:memory | Running check...
2023-01-27 16:38:14 UTC | CORE | DEBUG | (pkg/collector/worker/check_logger.go:59 in CheckFinished) | check:memory | Done running check
2023-01-27 16:38:14 UTC | CORE | DEBUG | (pkg/util/kubernetes/kubelet/podwatcher.go:147 in computeChanges) | Found 0 changed pods out of 11
2023-01-27 16:38:15 UTC | CORE | DEBUG | (pkg/logs/internal/launchers/file/launcher.go:141 in scan) | Scan - got 0 files from FilesToTail and currently tailing 0 files
2023-01-27 16:38:15 UTC | CORE | DEBUG | (pkg/logs/internal/launchers/file/launcher.go:195 in scan) | After stopping tailers, there are 0 tailers running.
2023-01-27 16:38:15 UTC | CORE | DEBUG | (pkg/logs/internal/launchers/file/launcher.go:212 in scan) | After starting new tailers, there are 0 tailers running. Limit is 500.
2023-01-27 16:38:15 UTC | CORE | DEBUG | (pkg/collector/worker/check_logger.go:42 in CheckStarted) | check:io | Running check...
2023-01-27 16:38:15 UTC | CORE | DEBUG | (pkg/collector/worker/check_logger.go:59 in CheckFinished) | check:io | Done running check
2023-01-27 16:38:16 UTC | CORE | DEBUG | (pkg/collector/worker/check_logger.go:42 in CheckStarted) | check:docker | Running check...
2023-01-27 16:38:16 UTC | CORE | DEBUG | (pkg/collector/corechecks/containers/generic/processor.go:116 in Run) | OpenFiles count for: &{{container 59f8054dd8439dda298171fecb121928e077dd0e81835fdfbbd86dd47be7e925} {agent  map[] map[annotation.io.kubernetes.container.hash:b7d4b3a9 annotation.io.kubernetes.container.ports:[{"name":"dogstatsdport","containerPort":8125,"protocol":"UDP"}] annotation.io.kubernetes.container.restartCount:0 annotation.io.kubernetes.container.terminationMessagePath:/dev/termination-log annotation.io.kubernetes.container.terminationMessagePolicy:File annotation.io.kubernetes.pod.terminationGracePeriod:30 io.kubernetes.container.logpath:/var/log/pods/default_datadog-lzcr8_3532449e-51ba-4864-a468-d7a4da5b094f/agent/0.log io.kubernetes.container.name:agent io.kubernetes.docker.type:container io.kubernetes.pod.name:datadog-lzcr8 io.kubernetes.pod.namespace:default io.kubernetes.pod.uid:3532449e-51ba-4864-a468-d7a4da5b094f io.kubernetes.sandbox.id:76d45bcbf4a6c61636d13d5c4c30ac05dcb850055dc7eed90a54e246fe6695bc maintainer:Datadog <package@datadoghq.com>]} map[CURL_CA_BUNDLE:/opt/datadog-agent/embedded/ssl/certs/cacert.pem DATADOG_CLUSTER_AGENT_ADMISSION_CONTROLLER_PORT:tcp://10.98.222.107:443 DATADOG_CLUSTER_AGENT_ADMISSION_CONTROLLER_PORT_443_TCP:tcp://10.98.222.107:443 DATADOG_CLUSTER_AGENT_ADMISSION_CONTROLLER_PORT_443_TCP_ADDR:10.98.222.107 DATADOG_CLUSTER_AGENT_ADMISSION_CONTROLLER_PORT_443_TCP_PORT:443 DATADOG_CLUSTER_AGENT_ADMISSION_CONTROLLER_PORT_443_TCP_PROTO:tcp DATADOG_CLUSTER_AGENT_ADMISSION_CONTROLLER_SERVICE_HOST:10.98.222.107 DATADOG_CLUSTER_AGENT_ADMISSION_CONTROLLER_SERVICE_PORT:443 DATADOG_CLUSTER_AGENT_PORT:tcp://10.104.229.207:5005 DATADOG_CLUSTER_AGENT_PORT_5005_TCP:tcp://10.104.229.207:5005 DATADOG_CLUSTER_AGENT_PORT_5005_TCP_ADDR:10.104.229.207 DATADOG_CLUSTER_AGENT_PORT_5005_TCP_PORT:5005 DATADOG_CLUSTER_AGENT_PORT_5005_TCP_PROTO:tcp DATADOG_CLUSTER_AGENT_SERVICE_HOST:10.104.229.207 DATADOG_CLUSTER_AGENT_SERVICE_PORT:5005 DATADOG_CLUSTER_AGENT_SERVICE_PORT_AGENTPORT:5005 DATADOG_PORT:udp://10.96.246.183:8125 DATADOG_PORT_8125_UDP:udp://10.96.246.183:8125 DATADOG_PORT_8125_UDP_ADDR:10.96.246.183 DATADOG_PORT_8125_UDP_PORT:8125 DATADOG_PORT_8125_UDP_PROTO:udp DATADOG_SERVICE_HOST:10.96.246.183 DATADOG_SERVICE_PORT:8125 DATADOG_SERVICE_PORT_DOGSTATSD:8125 DD_API_KEY:"***************************08bfc" DD_APM_ENABLED:false DD_CLUSTER_AGENT_AUTH_TOKEN:4tcpr3fNUzQmp1frQEqVx8EOLe8LN2fP DD_CLUSTER_AGENT_ENABLED:true DD_CLUSTER_AGENT_KUBERNETES_SERVICE_NAME:datadog-cluster-agent DD_CLUSTER_NAME:mini DD_DOGSTATSD_NON_LOCAL_TRAFFIC:true DD_DOGSTATSD_PORT:8125 DD_DOGSTATSD_SOCKET:/var/run/datadog/dsd.socket DD_EXPVAR_PORT:6000 DD_EXTRA_CONFIG_PROVIDERS:clusterchecks endpointschecks DD_HEALTH_PORT:5555 DD_IGNORE_AUTOCONF:etcd kube_controller_manager kube_scheduler kubernetes_state DD_KUBELET_TLS_VERIFY:false DD_KUBERNETES_KUBELET_HOST:192.168.49.2 DD_LOGS_CONFIG_AUTO_MULTI_LINE_DETECTION:false DD_LOGS_CONFIG_CONTAINER_COLLECT_ALL:false DD_LOGS_CONFIG_K8S_CONTAINER_USE_FILE:true DD_LOGS_ENABLED:true DD_LOG_LEVEL:INFO DD_PYTHON_VERSION:3 DD_SECRET_BACKEND_ARGUMENTS:/etc/secret-volume DD_SECRET_BACKEND_COMMAND:/readsecret_multiple_providers.sh DD_SECRET_BACKEND_COMMAND_ALLOW_GROUP_EXEC_PERM:true DEBIAN_FRONTEND:noninteractive DOCKER_DD_AGENT:true GODEBUG:x509ignoreCN=0 KUBERNETES:yes KUBERNETES_PORT:tcp://10.96.0.1:443 KUBERNETES_PORT_443_TCP:tcp://10.96.0.1:443 KUBERNETES_PORT_443_TCP_ADDR:10.96.0.1 KUBERNETES_PORT_443_TCP_PORT:443 KUBERNETES_PORT_443_TCP_PROTO:tcp KUBERNETES_SERVICE_HOST:10.96.0.1 KUBERNETES_SERVICE_PORT:443 KUBERNETES_SERVICE_PORT_HTTPS:443 LD_PRELOAD:/lib/x86_64-linux-gnu/nosys.so PATH:/opt/datadog-agent/bin/agent/:/opt/datadog-agent/embedded/bin/:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin S6_BEHAVIOUR_IF_STAGE2_FAILS:2 S6_KEEP_ENV:1 S6_LOGGING:0 S6_READ_ONLY_ROOT:1] datadog-lzcr8 {docker-pullable:********@sha256:4992b38387ca6dbb732f3493ed66cdf5efe7e08154b7e9e4159b1735dbaabe3f gcr.io/datadoghq/agent:7.41.1 gcr.io/datadoghq/agent agent 7.41.1} map[] 3317261 [{dogstatsdport 8125 UDP} { 8125 udp} { 8126 tcp}] docker {true running unknown 2023-01-27 16:32:21.734951171 +0000 UTC 2023-01-27 16:32:21.838310713 +0000 UTC 0001-01-01 00:00:00 +0000 UTC <nil>} []} not available through collector "system", err: unable to read any PID open FDs for cgroup id: 59f8054dd8439dda298171fecb121928e077dd0e81835fdfbbd86dd47be7e925. Unable to get count of open files
2023-01-27 16:38:16 UTC | CORE | DEBUG | (pkg/collector/corechecks/containers/docker/check_network.go:95 in Process) | Gathering network metrics for container: &{{container 59f8054dd8439dda298171fecb121928e077dd0e81835fdfbbd86dd47be7e925} {agent  map[] map[annotation.io.kubernetes.container.hash:b7d4b3a9 annotation.io.kubernetes.container.ports:[{"name":"dogstatsdport","containerPort":8125,"protocol":"UDP"}] annotation.io.kubernetes.container.restartCount:0 annotation.io.kubernetes.container.terminationMessagePath:/dev/termination-log annotation.io.kubernetes.container.terminationMessagePolicy:File annotation.io.kubernetes.pod.terminationGracePeriod:30 io.kubernetes.container.logpath:/var/log/pods/default_datadog-lzcr8_3532449e-51ba-4864-a468-d7a4da5b094f/agent/0.log io.kubernetes.container.name:agent io.kubernetes.docker.type:container io.kubernetes.pod.name:datadog-lzcr8 io.kubernetes.pod.namespace:default io.kubernetes.pod.uid:3532449e-51ba-4864-a468-d7a4da5b094f io.kubernetes.sandbox.id:76d45bcbf4a6c61636d13d5c4c30ac05dcb850055dc7eed90a54e246fe6695bc maintainer:Datadog <package@datadoghq.com>]} map[CURL_CA_BUNDLE:/opt/datadog-agent/embedded/ssl/certs/cacert.pem DATADOG_CLUSTER_AGENT_ADMISSION_CONTROLLER_PORT:tcp://10.98.222.107:443 DATADOG_CLUSTER_AGENT_ADMISSION_CONTROLLER_PORT_443_TCP:tcp://10.98.222.107:443 DATADOG_CLUSTER_AGENT_ADMISSION_CONTROLLER_PORT_443_TCP_ADDR:10.98.222.107 DATADOG_CLUSTER_AGENT_ADMISSION_CONTROLLER_PORT_443_TCP_PORT:443 DATADOG_CLUSTER_AGENT_ADMISSION_CONTROLLER_PORT_443_TCP_PROTO:tcp DATADOG_CLUSTER_AGENT_ADMISSION_CONTROLLER_SERVICE_HOST:10.98.222.107 DATADOG_CLUSTER_AGENT_ADMISSION_CONTROLLER_SERVICE_PORT:443 DATADOG_CLUSTER_AGENT_PORT:tcp://10.104.229.207:5005 DATADOG_CLUSTER_AGENT_PORT_5005_TCP:tcp://10.104.229.207:5005 DATADOG_CLUSTER_AGENT_PORT_5005_TCP_ADDR:10.104.229.207 DATADOG_CLUSTER_AGENT_PORT_5005_TCP_PORT:5005 DATADOG_CLUSTER_AGENT_PORT_5005_TCP_PROTO:tcp DATADOG_CLUSTER_AGENT_SERVICE_HOST:10.104.229.207 DATADOG_CLUSTER_AGENT_SERVICE_PORT:5005 DATADOG_CLUSTER_AGENT_SERVICE_PORT_AGENTPORT:5005 DATADOG_PORT:udp://10.96.246.183:8125 DATADOG_PORT_8125_UDP:udp://10.96.246.183:8125 DATADOG_PORT_8125_UDP_ADDR:10.96.246.183 DATADOG_PORT_8125_UDP_PORT:8125 DATADOG_PORT_8125_UDP_PROTO:udp DATADOG_SERVICE_HOST:10.96.246.183 DATADOG_SERVICE_PORT:8125 DATADOG_SERVICE_PORT_DOGSTATSD:8125 DD_API_KEY:"***************************08bfc" DD_APM_ENABLED:false DD_CLUSTER_AGENT_AUTH_TOKEN:4tcpr3fNUzQmp1frQEqVx8EOLe8LN2fP DD_CLUSTER_AGENT_ENABLED:true DD_CLUSTER_AGENT_KUBERNETES_SERVICE_NAME:datadog-cluster-agent DD_CLUSTER_NAME:mini DD_DOGSTATSD_NON_LOCAL_TRAFFIC:true DD_DOGSTATSD_PORT:8125 DD_DOGSTATSD_SOCKET:/var/run/datadog/dsd.socket DD_EXPVAR_PORT:6000 DD_EXTRA_CONFIG_PROVIDERS:clusterchecks endpointschecks DD_HEALTH_PORT:5555 DD_IGNORE_AUTOCONF:etcd kube_controller_manager kube_scheduler kubernetes_state DD_KUBELET_TLS_VERIFY:false DD_KUBERNETES_KUBELET_HOST:192.168.49.2 DD_LOGS_CONFIG_AUTO_MULTI_LINE_DETECTION:false DD_LOGS_CONFIG_CONTAINER_COLLECT_ALL:false DD_LOGS_CONFIG_K8S_CONTAINER_USE_FILE:true DD_LOGS_ENABLED:true DD_LOG_LEVEL:INFO DD_PYTHON_VERSION:3 DD_SECRET_BACKEND_ARGUMENTS:/etc/secret-volume DD_SECRET_BACKEND_COMMAND:/readsecret_multiple_providers.sh DD_SECRET_BACKEND_COMMAND_ALLOW_GROUP_EXEC_PERM:true DEBIAN_FRONTEND:noninteractive DOCKER_DD_AGENT:true GODEBUG:x509ignoreCN=0 KUBERNETES:yes KUBERNETES_PORT:tcp://10.96.0.1:443 KUBERNETES_PORT_443_TCP:tcp://10.96.0.1:443 KUBERNETES_PORT_443_TCP_ADDR:10.96.0.1 KUBERNETES_PORT_443_TCP_PORT:443 KUBERNETES_PORT_443_TCP_PROTO:tcp KUBERNETES_SERVICE_HOST:10.96.0.1 KUBERNETES_SERVICE_PORT:443 KUBERNETES_SERVICE_PORT_HTTPS:443 LD_PRELOAD:/lib/x86_64-linux-gnu/nosys.so PATH:/opt/datadog-agent/bin/agent/:/opt/datadog-agent/embedded/bin/:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin S6_BEHAVIOUR_IF_STAGE2_FAILS:2 S6_KEEP_ENV:1 S6_LOGGING:0 S6_READ_ONLY_ROOT:1] datadog-lzcr8 {docker-pullable:********@sha256:4992b38387ca6dbb732f3493ed66cdf5efe7e08154b7e9e4159b1735dbaabe3f gcr.io/datadoghq/agent:7.41.1 gcr.io/datadoghq/agent agent 7.41.1} map[] 3317261 [{dogstatsdport 8125 UDP} { 8125 udp} { 8126 tcp}] docker {true running unknown 2023-01-27 16:32:21.734951171 +0000 UTC 2023-01-27 16:32:21.838310713 +0000 UTC 0001-01-01 00:00:00 +0000 UTC <nil>} []} failed, metrics may be missing, err: no process found inside this cgroup, impossible to gather network stats
2023-01-27 16:38:16 UTC | CORE | DEBUG | (pkg/collector/corechecks/containers/docker/check_network.go:226 in findDockerNetworks) | No PID found for container: e20ff6db4165dfd5aabff2ed6f58b8f4fc1eae1174ca56e1204efe6ea7586bc1, skipping network
2023-01-27 16:38:16 UTC | CORE | DEBUG | (pkg/collector/corechecks/containers/docker/check_network.go:226 in findDockerNetworks) | No PID found for container: 76d45bcbf4a6c61636d13d5c4c30ac05dcb850055dc7eed90a54e246fe6695bc, skipping network
2023-01-27 16:38:16 UTC | CORE | DEBUG | (pkg/collector/corechecks/containers/docker/check_network.go:226 in findDockerNetworks) | No PID found for container: 8dd818c41c9f074b211c5c2bb6fc28d95247802845af0e8ff083a894d314313a, skipping network
2023-01-27 16:38:16 UTC | CORE | DEBUG | (pkg/collector/corechecks/containers/docker/check_network.go:205 in findDockerNetworks) | Container e8f8042ab20dab731de6c74db22687addb7e9529f8bd132e36f4596d0ebf418a is in network host mode, its network metrics are for the whole host
2023-01-27 16:38:16 UTC | CORE | DEBUG | (pkg/collector/corechecks/containers/docker/check_network.go:205 in findDockerNetworks) | Container f55f65aaa1b4bba2f21a9147e18b01bc0cb44ca7c789e2327525145e87b34be7 is in network host mode, its network metrics are for the whole host
2023-01-27 16:38:16 UTC | CORE | DEBUG | (pkg/collector/corechecks/containers/docker/check_network.go:226 in findDockerNetworks) | No PID found for container: a7eddfe2282b3779c113866c707e8c5ddd904f9e97ba107daeab92699d5c8d48, skipping network
2023-01-27 16:38:16 UTC | CORE | DEBUG | (pkg/collector/corechecks/containers/docker/check_network.go:226 in findDockerNetworks) | No PID found for container: 3a8b4db967a630c33d1e9556f11868a15de9b7141c24c03f70a9b05289229cbb, skipping network
2023-01-27 16:38:16 UTC | CORE | DEBUG | (pkg/collector/corechecks/containers/docker/check_network.go:205 in findDockerNetworks) | Container eaa983a0a2063c2c4135e206f8a78ea1ae5cbe130099e292733a8973cbfcd35a is in network host mode, its network metrics are for the whole host
2023-01-27 16:38:16 UTC | CORE | DEBUG | (pkg/collector/corechecks/containers/docker/check_network.go:205 in findDockerNetworks) | Container 5377f5f00a9e410217525941c79154ad0010a94179fd3d7fcc7fcb3ca9b331dd is in network host mode, its network metrics are for the whole host
2023-01-27 16:38:16 UTC | CORE | DEBUG | (pkg/collector/corechecks/containers/docker/check_network.go:205 in findDockerNetworks) | Container 94dfb30dd3bc339ce158d621fb9fd4c3b50feb7e4725609850a717fa2bf25fbb is in network host mode, its network metrics are for the whole host
2023-01-27 16:38:16 UTC | CORE | DEBUG | (pkg/collector/corechecks/containers/docker/check_network.go:205 in findDockerNetworks) | Container f7b4614499df44006c7a7b2fddd15d71e3b1af72c7447601e6bdb74a8485be3c is in network host mode, its network metrics are for the whole host
2023-01-27 16:38:16 UTC | CORE | DEBUG | (pkg/collector/worker/check_logger.go:59 in CheckFinished) | check:docker | Done running check
2023-01-27 16:38:16 UTC | CORE | DEBUG | (pkg/autodiscovery/config_poller.go:170 in collectOnce) | endpoints-checks provider: no configuration change
2023-01-27 16:38:16 UTC | CORE | DEBUG | (pkg/autodiscovery/config_poller.go:154 in poll) | No modifications in the templates stored in cluster-checks configuration provider
2023-01-27 16:38:17 UTC | CORE | DEBUG | (pkg/collector/worker/check_logger.go:42 in CheckStarted) | check:cpu | Running check...
2023-01-27 16:38:17 UTC | CORE | DEBUG | (pkg/collector/worker/check_logger.go:59 in CheckFinished) | check:cpu | Done running check
2023-01-27 16:38:18 UTC | CORE | DEBUG | (pkg/collector/worker/check_logger.go:42 in CheckStarted) | check:kube_apiserver_metrics | Running check...
2023-01-27 16:38:18 UTC | CORE | DEBUG | (pkg/collector/python/check.go:84 in runCheck) | Running python check kube_apiserver_metrics (version: '3.3.0', id: 'kube_apiserver_metrics:9870dafe14e0de84')
2023-01-27 16:38:18 UTC | CORE | DEBUG | (pkg/collector/worker/check_logger.go:59 in CheckFinished) | check:kube_apiserver_metrics | Done running check
2023-01-27 16:38:19 UTC | CORE | DEBUG | (pkg/collector/worker/check_logger.go:42 in CheckStarted) | check:datadog_cluster_agent | Running check...
2023-01-27 16:38:19 UTC | CORE | DEBUG | (pkg/collector/python/check.go:84 in runCheck) | Running python check datadog_cluster_agent (version: '2.4.0', id: 'datadog_cluster_agent:37b5fa98a5978a46')
2023-01-27 16:38:19 UTC | CORE | DEBUG | (pkg/collector/worker/check_logger.go:59 in CheckFinished) | check:datadog_cluster_agent | Done running check
2023-01-27 16:38:19 UTC | CORE | DEBUG | (pkg/util/kubernetes/kubelet/podwatcher.go:147 in computeChanges) | Found 0 changed pods out of 11
2023-01-27 16:38:21 UTC | CORE | DEBUG | (pkg/collector/worker/check_logger.go:42 in CheckStarted) | check:network | Running check...
2023-01-27 16:38:21 UTC | CORE | DEBUG | (pkg/collector/python/check.go:84 in runCheck) | Running python check network (version: '2.9.2', id: 'network:d884b5186b651429')
2023-01-27 16:38:21 UTC | CORE | DEBUG | (pkg/collector/worker/check_logger.go:59 in CheckFinished) | check:network | Done running check
2023-01-27 16:38:22 UTC | CORE | DEBUG | (pkg/collector/worker/check_logger.go:42 in CheckStarted) | check:load | Running check...
2023-01-27 16:38:22 UTC | CORE | DEBUG | (pkg/collector/worker/check_logger.go:59 in CheckFinished) | check:load | Done running check
2023-01-27 16:38:23 UTC | CORE | DEBUG | (pkg/collector/worker/check_logger.go:42 in CheckStarted) | check:file_handle | Running check...
2023-01-27 16:38:23 UTC | CORE | DEBUG | (pkg/collector/worker/check_logger.go:59 in CheckFinished) | check:file_handle | Done running check
2023-01-27 16:38:24 UTC | CORE | DEBUG | (pkg/collector/worker/check_logger.go:42 in CheckStarted) | check:disk | Running check...
2023-01-27 16:38:24 UTC | CORE | DEBUG | (pkg/collector/python/check.go:84 in runCheck) | Running python check disk (version: '4.7.1', id: 'disk:e5dffb8bef24336f')
2023-01-27 16:38:24 UTC | CORE | DEBUG | (pkg/collector/worker/check_logger.go:59 in CheckFinished) | check:disk | Done running check
2023-01-27 16:38:24 UTC | CORE | DEBUG | (pkg/config/config.go:1827 in IsCloudProviderEnabled) | cloud_provider_metadata is set to [aws gcp azure alibaba oracle ibm] in agent configuration, trying endpoints for AWS Cloud Provider
2023-01-27 16:38:24 UTC | CORE | DEBUG | (pkg/config/config.go:1827 in IsCloudProviderEnabled) | cloud_provider_metadata is set to [aws gcp azure alibaba oracle ibm] in agent configuration, trying endpoints for AWS Cloud Provider
2023-01-27 16:38:24 UTC | CORE | DEBUG | (pkg/config/config.go:1827 in IsCloudProviderEnabled) | cloud_provider_metadata is set to [aws gcp azure alibaba oracle ibm] in agent configuration, trying endpoints for AWS Cloud Provider
2023-01-27 16:38:24 UTC | CORE | DEBUG | (pkg/util/ecs/metadata/clients.go:65 in V1) | ECS metadata v1 client init error: temporary failure in ecsutil-meta-v1, will retry later: try delay not elapsed yet
2023-01-27 16:38:24 UTC | CORE | DEBUG | (pkg/workloadmeta/store.go:331 in startCandidates) | workloadmeta collector "ecs" could not start, but will retry. error: temporary failure in ecsutil-meta-v1, will retry later: try delay not elapsed yet
2023-01-27 16:38:24 UTC | CORE | DEBUG | (pkg/util/kubernetes/kubelet/podwatcher.go:147 in computeChanges) | Found 0 changed pods out of 11
2023-01-27 16:38:25 UTC | CORE | DEBUG | (pkg/aggregator/demultiplexer.go:144 in sendIterableSeries) | Demultiplexer: sendIterableSeries: start sending iterable series to the serializer
2023-01-27 16:38:25 UTC | CORE | DEBUG | (pkg/aggregator/aggregator.go:604 in sendServiceChecks) | Flushing 20 service checks to the forwarder
2023-01-27 16:38:25 UTC | CORE | DEBUG | (pkg/aggregator/aggregator.go:659 in sendEvents) | Flushing 1 events to the forwarder
2023-01-27 16:38:25 UTC | CORE | DEBUG | (pkg/aggregator/demultiplexer.go:150 in sendIterableSeries) | Demultiplexer: sendIterableSeries: stop routine
2023-01-27 16:38:25 UTC | CORE | DEBUG | (pkg/logs/internal/launchers/file/launcher.go:141 in scan) | Scan - got 0 files from FilesToTail and currently tailing 0 files
2023-01-27 16:38:25 UTC | CORE | DEBUG | (pkg/logs/internal/launchers/file/launcher.go:195 in scan) | After stopping tailers, there are 0 tailers running.
2023-01-27 16:38:25 UTC | CORE | DEBUG | (pkg/logs/internal/launchers/file/launcher.go:212 in scan) | After starting new tailers, there are 0 tailers running. Limit is 500.
2023-01-27 16:38:25 UTC | CORE | DEBUG | (pkg/collector/worker/check_logger.go:42 in CheckStarted) | check:container | Running check...
2023-01-27 16:38:25 UTC | CORE | DEBUG | (pkg/collector/corechecks/containers/generic/processor.go:116 in Run) | OpenFiles count for: &{{container 59f8054dd8439dda298171fecb121928e077dd0e81835fdfbbd86dd47be7e925} {agent  map[] map[annotation.io.kubernetes.container.hash:b7d4b3a9 annotation.io.kubernetes.container.ports:[{"name":"dogstatsdport","containerPort":8125,"protocol":"UDP"}] annotation.io.kubernetes.container.restartCount:0 annotation.io.kubernetes.container.terminationMessagePath:/dev/termination-log annotation.io.kubernetes.container.terminationMessagePolicy:File annotation.io.kubernetes.pod.terminationGracePeriod:30 io.kubernetes.container.logpath:/var/log/pods/default_datadog-lzcr8_3532449e-51ba-4864-a468-d7a4da5b094f/agent/0.log io.kubernetes.container.name:agent io.kubernetes.docker.type:container io.kubernetes.pod.name:datadog-lzcr8 io.kubernetes.pod.namespace:default io.kubernetes.pod.uid:3532449e-51ba-4864-a468-d7a4da5b094f io.kubernetes.sandbox.id:76d45bcbf4a6c61636d13d5c4c30ac05dcb850055dc7eed90a54e246fe6695bc maintainer:Datadog <package@datadoghq.com>]} map[CURL_CA_BUNDLE:/opt/datadog-agent/embedded/ssl/certs/cacert.pem DATADOG_CLUSTER_AGENT_ADMISSION_CONTROLLER_PORT:tcp://10.98.222.107:443 DATADOG_CLUSTER_AGENT_ADMISSION_CONTROLLER_PORT_443_TCP:tcp://10.98.222.107:443 DATADOG_CLUSTER_AGENT_ADMISSION_CONTROLLER_PORT_443_TCP_ADDR:10.98.222.107 DATADOG_CLUSTER_AGENT_ADMISSION_CONTROLLER_PORT_443_TCP_PORT:443 DATADOG_CLUSTER_AGENT_ADMISSION_CONTROLLER_PORT_443_TCP_PROTO:tcp DATADOG_CLUSTER_AGENT_ADMISSION_CONTROLLER_SERVICE_HOST:10.98.222.107 DATADOG_CLUSTER_AGENT_ADMISSION_CONTROLLER_SERVICE_PORT:443 DATADOG_CLUSTER_AGENT_PORT:tcp://10.104.229.207:5005 DATADOG_CLUSTER_AGENT_PORT_5005_TCP:tcp://10.104.229.207:5005 DATADOG_CLUSTER_AGENT_PORT_5005_TCP_ADDR:10.104.229.207 DATADOG_CLUSTER_AGENT_PORT_5005_TCP_PORT:5005 DATADOG_CLUSTER_AGENT_PORT_5005_TCP_PROTO:tcp DATADOG_CLUSTER_AGENT_SERVICE_HOST:10.104.229.207 DATADOG_CLUSTER_AGENT_SERVICE_PORT:5005 DATADOG_CLUSTER_AGENT_SERVICE_PORT_AGENTPORT:5005 DATADOG_PORT:udp://10.96.246.183:8125 DATADOG_PORT_8125_UDP:udp://10.96.246.183:8125 DATADOG_PORT_8125_UDP_ADDR:10.96.246.183 DATADOG_PORT_8125_UDP_PORT:8125 DATADOG_PORT_8125_UDP_PROTO:udp DATADOG_SERVICE_HOST:10.96.246.183 DATADOG_SERVICE_PORT:8125 DATADOG_SERVICE_PORT_DOGSTATSD:8125 DD_API_KEY:"***************************08bfc" DD_APM_ENABLED:false DD_CLUSTER_AGENT_AUTH_TOKEN:4tcpr3fNUzQmp1frQEqVx8EOLe8LN2fP DD_CLUSTER_AGENT_ENABLED:true DD_CLUSTER_AGENT_KUBERNETES_SERVICE_NAME:datadog-cluster-agent DD_CLUSTER_NAME:mini DD_DOGSTATSD_NON_LOCAL_TRAFFIC:true DD_DOGSTATSD_PORT:8125 DD_DOGSTATSD_SOCKET:/var/run/datadog/dsd.socket DD_EXPVAR_PORT:6000 DD_EXTRA_CONFIG_PROVIDERS:clusterchecks endpointschecks DD_HEALTH_PORT:5555 DD_IGNORE_AUTOCONF:etcd kube_controller_manager kube_scheduler kubernetes_state DD_KUBELET_TLS_VERIFY:false DD_KUBERNETES_KUBELET_HOST:192.168.49.2 DD_LOGS_CONFIG_AUTO_MULTI_LINE_DETECTION:false DD_LOGS_CONFIG_CONTAINER_COLLECT_ALL:false DD_LOGS_CONFIG_K8S_CONTAINER_USE_FILE:true DD_LOGS_ENABLED:true DD_LOG_LEVEL:INFO DD_PYTHON_VERSION:3 DD_SECRET_BACKEND_ARGUMENTS:/etc/secret-volume DD_SECRET_BACKEND_COMMAND:/readsecret_multiple_providers.sh DD_SECRET_BACKEND_COMMAND_ALLOW_GROUP_EXEC_PERM:true DEBIAN_FRONTEND:noninteractive DOCKER_DD_AGENT:true GODEBUG:x509ignoreCN=0 KUBERNETES:yes KUBERNETES_PORT:tcp://10.96.0.1:443 KUBERNETES_PORT_443_TCP:tcp://10.96.0.1:443 KUBERNETES_PORT_443_TCP_ADDR:10.96.0.1 KUBERNETES_PORT_443_TCP_PORT:443 KUBERNETES_PORT_443_TCP_PROTO:tcp KUBERNETES_SERVICE_HOST:10.96.0.1 KUBERNETES_SERVICE_PORT:443 KUBERNETES_SERVICE_PORT_HTTPS:443 LD_PRELOAD:/lib/x86_64-linux-gnu/nosys.so PATH:/opt/datadog-agent/bin/agent/:/opt/datadog-agent/embedded/bin/:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin S6_BEHAVIOUR_IF_STAGE2_FAILS:2 S6_KEEP_ENV:1 S6_LOGGING:0 S6_READ_ONLY_ROOT:1] datadog-lzcr8 {docker-pullable:********@sha256:4992b38387ca6dbb732f3493ed66cdf5efe7e08154b7e9e4159b1735dbaabe3f gcr.io/datadoghq/agent:7.41.1 gcr.io/datadoghq/agent agent 7.41.1} map[] 3317261 [{dogstatsdport 8125 UDP} { 8125 udp} { 8126 tcp}] docker {true running unknown 2023-01-27 16:32:21.734951171 +0000 UTC 2023-01-27 16:32:21.838310713 +0000 UTC 0001-01-01 00:00:00 +0000 UTC <nil>} []} not available through collector "system", err: unable to read any PID open FDs for cgroup id: 59f8054dd8439dda298171fecb121928e077dd0e81835fdfbbd86dd47be7e925. Unable to get count of open files
2023-01-27 16:38:25 UTC | CORE | DEBUG | (pkg/collector/corechecks/containers/generic/processor_network.go:50 in Process) | Gathering network metrics for container: &{{container 59f8054dd8439dda298171fecb121928e077dd0e81835fdfbbd86dd47be7e925} {agent  map[] map[annotation.io.kubernetes.container.hash:b7d4b3a9 annotation.io.kubernetes.container.ports:[{"name":"dogstatsdport","containerPort":8125,"protocol":"UDP"}] annotation.io.kubernetes.container.restartCount:0 annotation.io.kubernetes.container.terminationMessagePath:/dev/termination-log annotation.io.kubernetes.container.terminationMessagePolicy:File annotation.io.kubernetes.pod.terminationGracePeriod:30 io.kubernetes.container.logpath:/var/log/pods/default_datadog-lzcr8_3532449e-51ba-4864-a468-d7a4da5b094f/agent/0.log io.kubernetes.container.name:agent io.kubernetes.docker.type:container io.kubernetes.pod.name:datadog-lzcr8 io.kubernetes.pod.namespace:default io.kubernetes.pod.uid:3532449e-51ba-4864-a468-d7a4da5b094f io.kubernetes.sandbox.id:76d45bcbf4a6c61636d13d5c4c30ac05dcb850055dc7eed90a54e246fe6695bc maintainer:Datadog <package@datadoghq.com>]} map[CURL_CA_BUNDLE:/opt/datadog-agent/embedded/ssl/certs/cacert.pem DATADOG_CLUSTER_AGENT_ADMISSION_CONTROLLER_PORT:tcp://10.98.222.107:443 DATADOG_CLUSTER_AGENT_ADMISSION_CONTROLLER_PORT_443_TCP:tcp://10.98.222.107:443 DATADOG_CLUSTER_AGENT_ADMISSION_CONTROLLER_PORT_443_TCP_ADDR:10.98.222.107 DATADOG_CLUSTER_AGENT_ADMISSION_CONTROLLER_PORT_443_TCP_PORT:443 DATADOG_CLUSTER_AGENT_ADMISSION_CONTROLLER_PORT_443_TCP_PROTO:tcp DATADOG_CLUSTER_AGENT_ADMISSION_CONTROLLER_SERVICE_HOST:10.98.222.107 DATADOG_CLUSTER_AGENT_ADMISSION_CONTROLLER_SERVICE_PORT:443 DATADOG_CLUSTER_AGENT_PORT:tcp://10.104.229.207:5005 DATADOG_CLUSTER_AGENT_PORT_5005_TCP:tcp://10.104.229.207:5005 DATADOG_CLUSTER_AGENT_PORT_5005_TCP_ADDR:10.104.229.207 DATADOG_CLUSTER_AGENT_PORT_5005_TCP_PORT:5005 DATADOG_CLUSTER_AGENT_PORT_5005_TCP_PROTO:tcp DATADOG_CLUSTER_AGENT_SERVICE_HOST:10.104.229.207 DATADOG_CLUSTER_AGENT_SERVICE_PORT:5005 DATADOG_CLUSTER_AGENT_SERVICE_PORT_AGENTPORT:5005 DATADOG_PORT:udp://10.96.246.183:8125 DATADOG_PORT_8125_UDP:udp://10.96.246.183:8125 DATADOG_PORT_8125_UDP_ADDR:10.96.246.183 DATADOG_PORT_8125_UDP_PORT:8125 DATADOG_PORT_8125_UDP_PROTO:udp DATADOG_SERVICE_HOST:10.96.246.183 DATADOG_SERVICE_PORT:8125 DATADOG_SERVICE_PORT_DOGSTATSD:8125 DD_API_KEY:"***************************08bfc" DD_APM_ENABLED:false DD_CLUSTER_AGENT_AUTH_TOKEN:4tcpr3fNUzQmp1frQEqVx8EOLe8LN2fP DD_CLUSTER_AGENT_ENABLED:true DD_CLUSTER_AGENT_KUBERNETES_SERVICE_NAME:datadog-cluster-agent DD_CLUSTER_NAME:mini DD_DOGSTATSD_NON_LOCAL_TRAFFIC:true DD_DOGSTATSD_PORT:8125 DD_DOGSTATSD_SOCKET:/var/run/datadog/dsd.socket DD_EXPVAR_PORT:6000 DD_EXTRA_CONFIG_PROVIDERS:clusterchecks endpointschecks DD_HEALTH_PORT:5555 DD_IGNORE_AUTOCONF:etcd kube_controller_manager kube_scheduler kubernetes_state DD_KUBELET_TLS_VERIFY:false DD_KUBERNETES_KUBELET_HOST:192.168.49.2 DD_LOGS_CONFIG_AUTO_MULTI_LINE_DETECTION:false DD_LOGS_CONFIG_CONTAINER_COLLECT_ALL:false DD_LOGS_CONFIG_K8S_CONTAINER_USE_FILE:true DD_LOGS_ENABLED:true DD_LOG_LEVEL:INFO DD_PYTHON_VERSION:3 DD_SECRET_BACKEND_ARGUMENTS:/etc/secret-volume DD_SECRET_BACKEND_COMMAND:/readsecret_multiple_providers.sh DD_SECRET_BACKEND_COMMAND_ALLOW_GROUP_EXEC_PERM:true DEBIAN_FRONTEND:noninteractive DOCKER_DD_AGENT:true GODEBUG:x509ignoreCN=0 KUBERNETES:yes KUBERNETES_PORT:tcp://10.96.0.1:443 KUBERNETES_PORT_443_TCP:tcp://10.96.0.1:443 KUBERNETES_PORT_443_TCP_ADDR:10.96.0.1 KUBERNETES_PORT_443_TCP_PORT:443 KUBERNETES_PORT_443_TCP_PROTO:tcp KUBERNETES_SERVICE_HOST:10.96.0.1 KUBERNETES_SERVICE_PORT:443 KUBERNETES_SERVICE_PORT_HTTPS:443 LD_PRELOAD:/lib/x86_64-linux-gnu/nosys.so PATH:/opt/datadog-agent/bin/agent/:/opt/datadog-agent/embedded/bin/:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin S6_BEHAVIOUR_IF_STAGE2_FAILS:2 S6_KEEP_ENV:1 S6_LOGGING:0 S6_READ_ONLY_ROOT:1] datadog-lzcr8 {docker-pullable:********@sha256:4992b38387ca6dbb732f3493ed66cdf5efe7e08154b7e9e4159b1735dbaabe3f gcr.io/datadoghq/agent:7.41.1 gcr.io/datadoghq/agent agent 7.41.1} map[] 3317261 [{dogstatsdport 8125 UDP} { 8125 udp} { 8126 tcp}] docker {true running unknown 2023-01-27 16:32:21.734951171 +0000 UTC 2023-01-27 16:32:21.838310713 +0000 UTC 0001-01-01 00:00:00 +0000 UTC <nil>} []} failed, metrics may be missing, err: no process found inside this cgroup, impossible to gather network stats
2023-01-27 16:38:25 UTC | CORE | DEBUG | (pkg/collector/worker/check_logger.go:59 in CheckFinished) | check:container | Done running check
2023-01-27 16:38:26 UTC | CORE | DEBUG | (pkg/collector/worker/check_logger.go:42 in CheckStarted) | check:kubelet | Running check...
2023-01-27 16:38:26 UTC | CORE | DEBUG | (pkg/collector/python/check.go:84 in runCheck) | Running python check kubelet (version: '7.5.1', id: 'kubelet:5bbc63f3938c02f4')
2023-01-27 16:38:26 UTC | CORE | DEBUG | (pkg/collector/python/kubeutil.go:62 in GetKubeletConnectionInfo) | cache hit for kubelet connection info
2023-01-27 16:38:26 UTC | CORE | DEBUG | (pkg/autodiscovery/config_poller.go:170 in collectOnce) | endpoints-checks provider: no configuration change
2023-01-27 16:38:26 UTC | CORE | DEBUG | (pkg/autodiscovery/config_poller.go:154 in poll) | No modifications in the templates stored in cluster-checks configuration provider
2023-01-27 16:38:27 UTC | CORE | DEBUG | (pkg/collector/worker/check_logger.go:42 in CheckStarted) | check:coredns | Running check...
2023-01-27 16:38:27 UTC | CORE | DEBUG | (pkg/collector/python/check.go:84 in runCheck) | Running python check coredns (version: '2.3.0', id: 'coredns:435d3ef1baaaec95')
2023-01-27 16:38:27 UTC | CORE | DEBUG | (pkg/collector/worker/check_logger.go:59 in CheckFinished) | check:coredns | Done running check
2023-01-27 16:38:28 UTC | CORE | DEBUG | (pkg/collector/worker/check_logger.go:42 in CheckStarted) | check:uptime | Running check...
2023-01-27 16:38:28 UTC | CORE | DEBUG | (pkg/collector/worker/check_logger.go:59 in CheckFinished) | check:uptime | Done running check
2023-01-27 16:38:29 UTC | CORE | DEBUG | (pkg/collector/worker/check_logger.go:42 in CheckStarted) | check:memory | Running check...
2023-01-27 16:38:29 UTC | CORE | DEBUG | (pkg/collector/worker/check_logger.go:59 in CheckFinished) | check:memory | Done running check
2023-01-27 16:38:29 UTC | CORE | DEBUG | (pkg/util/kubernetes/kubelet/podwatcher.go:147 in computeChanges) | Found 0 changed pods out of 11
2023-01-27 16:38:30 UTC | CORE | DEBUG | (pkg/collector/worker/check_logger.go:42 in CheckStarted) | check:io | Running check...
2023-01-27 16:38:30 UTC | CORE | DEBUG | (pkg/collector/worker/check_logger.go:59 in CheckFinished) | check:io | Done running check
2023-01-27 16:38:30 UTC | CORE | DEBUG | (pkg/collector/worker/check_logger.go:59 in CheckFinished) | check:kubelet | Done running check
2023-01-27 16:38:31 UTC | CORE | DEBUG | (pkg/collector/worker/check_logger.go:42 in CheckStarted) | check:docker | Running check...
2023-01-27 16:38:31 UTC | CORE | DEBUG | (pkg/collector/corechecks/containers/generic/processor.go:116 in Run) | OpenFiles count for: &{{container 59f8054dd8439dda298171fecb121928e077dd0e81835fdfbbd86dd47be7e925} {agent  map[] map[annotation.io.kubernetes.container.hash:b7d4b3a9 annotation.io.kubernetes.container.ports:[{"name":"dogstatsdport","containerPort":8125,"protocol":"UDP"}] annotation.io.kubernetes.container.restartCount:0 annotation.io.kubernetes.container.terminationMessagePath:/dev/termination-log annotation.io.kubernetes.container.terminationMessagePolicy:File annotation.io.kubernetes.pod.terminationGracePeriod:30 io.kubernetes.container.logpath:/var/log/pods/default_datadog-lzcr8_3532449e-51ba-4864-a468-d7a4da5b094f/agent/0.log io.kubernetes.container.name:agent io.kubernetes.docker.type:container io.kubernetes.pod.name:datadog-lzcr8 io.kubernetes.pod.namespace:default io.kubernetes.pod.uid:3532449e-51ba-4864-a468-d7a4da5b094f io.kubernetes.sandbox.id:76d45bcbf4a6c61636d13d5c4c30ac05dcb850055dc7eed90a54e246fe6695bc maintainer:Datadog <package@datadoghq.com>]} map[CURL_CA_BUNDLE:/opt/datadog-agent/embedded/ssl/certs/cacert.pem DATADOG_CLUSTER_AGENT_ADMISSION_CONTROLLER_PORT:tcp://10.98.222.107:443 DATADOG_CLUSTER_AGENT_ADMISSION_CONTROLLER_PORT_443_TCP:tcp://10.98.222.107:443 DATADOG_CLUSTER_AGENT_ADMISSION_CONTROLLER_PORT_443_TCP_ADDR:10.98.222.107 DATADOG_CLUSTER_AGENT_ADMISSION_CONTROLLER_PORT_443_TCP_PORT:443 DATADOG_CLUSTER_AGENT_ADMISSION_CONTROLLER_PORT_443_TCP_PROTO:tcp DATADOG_CLUSTER_AGENT_ADMISSION_CONTROLLER_SERVICE_HOST:10.98.222.107 DATADOG_CLUSTER_AGENT_ADMISSION_CONTROLLER_SERVICE_PORT:443 DATADOG_CLUSTER_AGENT_PORT:tcp://10.104.229.207:5005 DATADOG_CLUSTER_AGENT_PORT_5005_TCP:tcp://10.104.229.207:5005 DATADOG_CLUSTER_AGENT_PORT_5005_TCP_ADDR:10.104.229.207 DATADOG_CLUSTER_AGENT_PORT_5005_TCP_PORT:5005 DATADOG_CLUSTER_AGENT_PORT_5005_TCP_PROTO:tcp DATADOG_CLUSTER_AGENT_SERVICE_HOST:10.104.229.207 DATADOG_CLUSTER_AGENT_SERVICE_PORT:5005 DATADOG_CLUSTER_AGENT_SERVICE_PORT_AGENTPORT:5005 DATADOG_PORT:udp://10.96.246.183:8125 DATADOG_PORT_8125_UDP:udp://10.96.246.183:8125 DATADOG_PORT_8125_UDP_ADDR:10.96.246.183 DATADOG_PORT_8125_UDP_PORT:8125 DATADOG_PORT_8125_UDP_PROTO:udp DATADOG_SERVICE_HOST:10.96.246.183 DATADOG_SERVICE_PORT:8125 DATADOG_SERVICE_PORT_DOGSTATSD:8125 DD_API_KEY:"***************************08bfc" DD_APM_ENABLED:false DD_CLUSTER_AGENT_AUTH_TOKEN:4tcpr3fNUzQmp1frQEqVx8EOLe8LN2fP DD_CLUSTER_AGENT_ENABLED:true DD_CLUSTER_AGENT_KUBERNETES_SERVICE_NAME:datadog-cluster-agent DD_CLUSTER_NAME:mini DD_DOGSTATSD_NON_LOCAL_TRAFFIC:true DD_DOGSTATSD_PORT:8125 DD_DOGSTATSD_SOCKET:/var/run/datadog/dsd.socket DD_EXPVAR_PORT:6000 DD_EXTRA_CONFIG_PROVIDERS:clusterchecks endpointschecks DD_HEALTH_PORT:5555 DD_IGNORE_AUTOCONF:etcd kube_controller_manager kube_scheduler kubernetes_state DD_KUBELET_TLS_VERIFY:false DD_KUBERNETES_KUBELET_HOST:192.168.49.2 DD_LOGS_CONFIG_AUTO_MULTI_LINE_DETECTION:false DD_LOGS_CONFIG_CONTAINER_COLLECT_ALL:false DD_LOGS_CONFIG_K8S_CONTAINER_USE_FILE:true DD_LOGS_ENABLED:true DD_LOG_LEVEL:INFO DD_PYTHON_VERSION:3 DD_SECRET_BACKEND_ARGUMENTS:/etc/secret-volume DD_SECRET_BACKEND_COMMAND:/readsecret_multiple_providers.sh DD_SECRET_BACKEND_COMMAND_ALLOW_GROUP_EXEC_PERM:true DEBIAN_FRONTEND:noninteractive DOCKER_DD_AGENT:true GODEBUG:x509ignoreCN=0 KUBERNETES:yes KUBERNETES_PORT:tcp://10.96.0.1:443 KUBERNETES_PORT_443_TCP:tcp://10.96.0.1:443 KUBERNETES_PORT_443_TCP_ADDR:10.96.0.1 KUBERNETES_PORT_443_TCP_PORT:443 KUBERNETES_PORT_443_TCP_PROTO:tcp KUBERNETES_SERVICE_HOST:10.96.0.1 KUBERNETES_SERVICE_PORT:443 KUBERNETES_SERVICE_PORT_HTTPS:443 LD_PRELOAD:/lib/x86_64-linux-gnu/nosys.so PATH:/opt/datadog-agent/bin/agent/:/opt/datadog-agent/embedded/bin/:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin S6_BEHAVIOUR_IF_STAGE2_FAILS:2 S6_KEEP_ENV:1 S6_LOGGING:0 S6_READ_ONLY_ROOT:1] datadog-lzcr8 {docker-pullable:********@sha256:4992b38387ca6dbb732f3493ed66cdf5efe7e08154b7e9e4159b1735dbaabe3f gcr.io/datadoghq/agent:7.41.1 gcr.io/datadoghq/agent agent 7.41.1} map[] 3317261 [{dogstatsdport 8125 UDP} { 8125 udp} { 8126 tcp}] docker {true running unknown 2023-01-27 16:32:21.734951171 +0000 UTC 2023-01-27 16:32:21.838310713 +0000 UTC 0001-01-01 00:00:00 +0000 UTC <nil>} []} not available through collector "system", err: unable to read any PID open FDs for cgroup id: 59f8054dd8439dda298171fecb121928e077dd0e81835fdfbbd86dd47be7e925. Unable to get count of open files
2023-01-27 16:38:31 UTC | CORE | DEBUG | (pkg/collector/corechecks/containers/docker/check_network.go:95 in Process) | Gathering network metrics for container: &{{container 59f8054dd8439dda298171fecb121928e077dd0e81835fdfbbd86dd47be7e925} {agent  map[] map[annotation.io.kubernetes.container.hash:b7d4b3a9 annotation.io.kubernetes.container.ports:[{"name":"dogstatsdport","containerPort":8125,"protocol":"UDP"}] annotation.io.kubernetes.container.restartCount:0 annotation.io.kubernetes.container.terminationMessagePath:/dev/termination-log annotation.io.kubernetes.container.terminationMessagePolicy:File annotation.io.kubernetes.pod.terminationGracePeriod:30 io.kubernetes.container.logpath:/var/log/pods/default_datadog-lzcr8_3532449e-51ba-4864-a468-d7a4da5b094f/agent/0.log io.kubernetes.container.name:agent io.kubernetes.docker.type:container io.kubernetes.pod.name:datadog-lzcr8 io.kubernetes.pod.namespace:default io.kubernetes.pod.uid:3532449e-51ba-4864-a468-d7a4da5b094f io.kubernetes.sandbox.id:76d45bcbf4a6c61636d13d5c4c30ac05dcb850055dc7eed90a54e246fe6695bc maintainer:Datadog <package@datadoghq.com>]} map[CURL_CA_BUNDLE:/opt/datadog-agent/embedded/ssl/certs/cacert.pem DATADOG_CLUSTER_AGENT_ADMISSION_CONTROLLER_PORT:tcp://10.98.222.107:443 DATADOG_CLUSTER_AGENT_ADMISSION_CONTROLLER_PORT_443_TCP:tcp://10.98.222.107:443 DATADOG_CLUSTER_AGENT_ADMISSION_CONTROLLER_PORT_443_TCP_ADDR:10.98.222.107 DATADOG_CLUSTER_AGENT_ADMISSION_CONTROLLER_PORT_443_TCP_PORT:443 DATADOG_CLUSTER_AGENT_ADMISSION_CONTROLLER_PORT_443_TCP_PROTO:tcp DATADOG_CLUSTER_AGENT_ADMISSION_CONTROLLER_SERVICE_HOST:10.98.222.107 DATADOG_CLUSTER_AGENT_ADMISSION_CONTROLLER_SERVICE_PORT:443 DATADOG_CLUSTER_AGENT_PORT:tcp://10.104.229.207:5005 DATADOG_CLUSTER_AGENT_PORT_5005_TCP:tcp://10.104.229.207:5005 DATADOG_CLUSTER_AGENT_PORT_5005_TCP_ADDR:10.104.229.207 DATADOG_CLUSTER_AGENT_PORT_5005_TCP_PORT:5005 DATADOG_CLUSTER_AGENT_PORT_5005_TCP_PROTO:tcp DATADOG_CLUSTER_AGENT_SERVICE_HOST:10.104.229.207 DATADOG_CLUSTER_AGENT_SERVICE_PORT:5005 DATADOG_CLUSTER_AGENT_SERVICE_PORT_AGENTPORT:5005 DATADOG_PORT:udp://10.96.246.183:8125 DATADOG_PORT_8125_UDP:udp://10.96.246.183:8125 DATADOG_PORT_8125_UDP_ADDR:10.96.246.183 DATADOG_PORT_8125_UDP_PORT:8125 DATADOG_PORT_8125_UDP_PROTO:udp DATADOG_SERVICE_HOST:10.96.246.183 DATADOG_SERVICE_PORT:8125 DATADOG_SERVICE_PORT_DOGSTATSD:8125 DD_API_KEY:"***************************08bfc" DD_APM_ENABLED:false DD_CLUSTER_AGENT_AUTH_TOKEN:4tcpr3fNUzQmp1frQEqVx8EOLe8LN2fP DD_CLUSTER_AGENT_ENABLED:true DD_CLUSTER_AGENT_KUBERNETES_SERVICE_NAME:datadog-cluster-agent DD_CLUSTER_NAME:mini DD_DOGSTATSD_NON_LOCAL_TRAFFIC:true DD_DOGSTATSD_PORT:8125 DD_DOGSTATSD_SOCKET:/var/run/datadog/dsd.socket DD_EXPVAR_PORT:6000 DD_EXTRA_CONFIG_PROVIDERS:clusterchecks endpointschecks DD_HEALTH_PORT:5555 DD_IGNORE_AUTOCONF:etcd kube_controller_manager kube_scheduler kubernetes_state DD_KUBELET_TLS_VERIFY:false DD_KUBERNETES_KUBELET_HOST:192.168.49.2 DD_LOGS_CONFIG_AUTO_MULTI_LINE_DETECTION:false DD_LOGS_CONFIG_CONTAINER_COLLECT_ALL:false DD_LOGS_CONFIG_K8S_CONTAINER_USE_FILE:true DD_LOGS_ENABLED:true DD_LOG_LEVEL:INFO DD_PYTHON_VERSION:3 DD_SECRET_BACKEND_ARGUMENTS:/etc/secret-volume DD_SECRET_BACKEND_COMMAND:/readsecret_multiple_providers.sh DD_SECRET_BACKEND_COMMAND_ALLOW_GROUP_EXEC_PERM:true DEBIAN_FRONTEND:noninteractive DOCKER_DD_AGENT:true GODEBUG:x509ignoreCN=0 KUBERNETES:yes KUBERNETES_PORT:tcp://10.96.0.1:443 KUBERNETES_PORT_443_TCP:tcp://10.96.0.1:443 KUBERNETES_PORT_443_TCP_ADDR:10.96.0.1 KUBERNETES_PORT_443_TCP_PORT:443 KUBERNETES_PORT_443_TCP_PROTO:tcp KUBERNETES_SERVICE_HOST:10.96.0.1 KUBERNETES_SERVICE_PORT:443 KUBERNETES_SERVICE_PORT_HTTPS:443 LD_PRELOAD:/lib/x86_64-linux-gnu/nosys.so PATH:/opt/datadog-agent/bin/agent/:/opt/datadog-agent/embedded/bin/:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin S6_BEHAVIOUR_IF_STAGE2_FAILS:2 S6_KEEP_ENV:1 S6_LOGGING:0 S6_READ_ONLY_ROOT:1] datadog-lzcr8 {docker-pullable:********@sha256:4992b38387ca6dbb732f3493ed66cdf5efe7e08154b7e9e4159b1735dbaabe3f gcr.io/datadoghq/agent:7.41.1 gcr.io/datadoghq/agent agent 7.41.1} map[] 3317261 [{dogstatsdport 8125 UDP} { 8125 udp} { 8126 tcp}] docker {true running unknown 2023-01-27 16:32:21.734951171 +0000 UTC 2023-01-27 16:32:21.838310713 +0000 UTC 0001-01-01 00:00:00 +0000 UTC <nil>} []} failed, metrics may be missing, err: no process found inside this cgroup, impossible to gather network stats
2023-01-27 16:38:31 UTC | CORE | DEBUG | (pkg/collector/corechecks/containers/docker/check_network.go:226 in findDockerNetworks) | No PID found for container: e20ff6db4165dfd5aabff2ed6f58b8f4fc1eae1174ca56e1204efe6ea7586bc1, skipping network
2023-01-27 16:38:31 UTC | CORE | DEBUG | (pkg/collector/corechecks/containers/docker/check_network.go:226 in findDockerNetworks) | No PID found for container: 76d45bcbf4a6c61636d13d5c4c30ac05dcb850055dc7eed90a54e246fe6695bc, skipping network
2023-01-27 16:38:31 UTC | CORE | DEBUG | (pkg/collector/corechecks/containers/docker/check_network.go:226 in findDockerNetworks) | No PID found for container: 8dd818c41c9f074b211c5c2bb6fc28d95247802845af0e8ff083a894d314313a, skipping network
2023-01-27 16:38:31 UTC | CORE | DEBUG | (pkg/collector/corechecks/containers/docker/check_network.go:205 in findDockerNetworks) | Container e8f8042ab20dab731de6c74db22687addb7e9529f8bd132e36f4596d0ebf418a is in network host mode, its network metrics are for the whole host
2023-01-27 16:38:31 UTC | CORE | DEBUG | (pkg/collector/corechecks/containers/docker/check_network.go:205 in findDockerNetworks) | Container f55f65aaa1b4bba2f21a9147e18b01bc0cb44ca7c789e2327525145e87b34be7 is in network host mode, its network metrics are for the whole host
2023-01-27 16:38:31 UTC | CORE | DEBUG | (pkg/collector/corechecks/containers/docker/check_network.go:226 in findDockerNetworks) | No PID found for container: a7eddfe2282b3779c113866c707e8c5ddd904f9e97ba107daeab92699d5c8d48, skipping network
2023-01-27 16:38:31 UTC | CORE | DEBUG | (pkg/collector/corechecks/containers/docker/check_network.go:226 in findDockerNetworks) | No PID found for container: 3a8b4db967a630c33d1e9556f11868a15de9b7141c24c03f70a9b05289229cbb, skipping network
2023-01-27 16:38:31 UTC | CORE | DEBUG | (pkg/collector/corechecks/containers/docker/check_network.go:205 in findDockerNetworks) | Container eaa983a0a2063c2c4135e206f8a78ea1ae5cbe130099e292733a8973cbfcd35a is in network host mode, its network metrics are for the whole host
2023-01-27 16:38:31 UTC | CORE | DEBUG | (pkg/collector/corechecks/containers/docker/check_network.go:205 in findDockerNetworks) | Container 5377f5f00a9e410217525941c79154ad0010a94179fd3d7fcc7fcb3ca9b331dd is in network host mode, its network metrics are for the whole host
2023-01-27 16:38:31 UTC | CORE | DEBUG | (pkg/collector/corechecks/containers/docker/check_network.go:205 in findDockerNetworks) | Container 94dfb30dd3bc339ce158d621fb9fd4c3b50feb7e4725609850a717fa2bf25fbb is in network host mode, its network metrics are for the whole host
2023-01-27 16:38:31 UTC | CORE | DEBUG | (pkg/collector/corechecks/containers/docker/check_network.go:205 in findDockerNetworks) | Container f7b4614499df44006c7a7b2fddd15d71e3b1af72c7447601e6bdb74a8485be3c is in network host mode, its network metrics are for the whole host
2023-01-27 16:38:31 UTC | CORE | DEBUG | (pkg/collector/worker/check_logger.go:59 in CheckFinished) | check:docker | Done running check
2023-01-27 16:38:32 UTC | CORE | DEBUG | (pkg/collector/worker/check_logger.go:42 in CheckStarted) | check:cpu | Running check...
2023-01-27 16:38:32 UTC | CORE | DEBUG | (pkg/collector/worker/check_logger.go:59 in CheckFinished) | check:cpu | Done running check
2023-01-27 16:38:33 UTC | CORE | DEBUG | (pkg/collector/worker/check_logger.go:42 in CheckStarted) | check:kube_apiserver_metrics | Running check...
2023-01-27 16:38:33 UTC | CORE | DEBUG | (pkg/collector/python/check.go:84 in runCheck) | Running python check kube_apiserver_metrics (version: '3.3.0', id: 'kube_apiserver_metrics:9870dafe14e0de84')
2023-01-27 16:38:33 UTC | CORE | DEBUG | (pkg/collector/worker/check_logger.go:59 in CheckFinished) | check:kube_apiserver_metrics | Done running check
2023-01-27 16:38:34 UTC | CORE | DEBUG | (pkg/collector/worker/check_logger.go:42 in CheckStarted) | check:datadog_cluster_agent | Running check...
2023-01-27 16:38:34 UTC | CORE | DEBUG | (pkg/collector/python/check.go:84 in runCheck) | Running python check datadog_cluster_agent (version: '2.4.0', id: 'datadog_cluster_agent:37b5fa98a5978a46')
2023-01-27 16:38:34 UTC | CORE | DEBUG | (pkg/collector/worker/check_logger.go:59 in CheckFinished) | check:datadog_cluster_agent | Done running check
2023-01-27 16:38:34 UTC | CORE | DEBUG | (pkg/util/kubernetes/kubelet/podwatcher.go:147 in computeChanges) | Found 0 changed pods out of 11
